sql/tests: TestRandomSyntaxSQLSmith failed

sql/tests.TestRandomSyntaxSQLSmith [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11354755?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11354755?buildTab=artifacts#/) on master @ [673bde530075ed690cf115aa0a5bdd7191d63f54](https://github.com/cockroachdb/cockroach/commits/673bde530075ed690cf115aa0a5bdd7191d63f54):

Random syntax error:

```
    rsg_test.go:871: Crash detected: server panic: statement exec timeout
```
Query:

```
        CREATE FUNCTION "\\U000AF3C9func111"(IN p0 BIT(32))
        	RETURNS TIMETZ
        	VOLATILE
        	NOT LEAKPROOF
        	LANGUAGE SQL
        	AS $funcbody$SELECT tab2760.col3_3 AS "co""l6687", 'null':::JSONB AS "col 6688", B'01100010110000000110001011011001' AS col6689 FROM defaultdb.public."tabl%qe3"@[0] AS tab2760 ORDER BY tab2760.col3_0 DESC NULLS FIRST, tab2760.crdb_internal_mvcc_timestamp NULLS FIRST LIMIT 20:::INT8;
```
Schema:

```
    rsg_test.go:681: To reproduce, use schema:
    rsg_test.go:683: SET sql_safe_updates = false;;
    rsg_test.go:683: SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;;
    rsg_test.go:683: SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;;
    rsg_test.go:683: CREATE TABLE table1 (_col1_0 TSQUERY NOT NULL, "co,l1_1" BOX2D NOT NULL, col1_2 DATE NOT NULL, "co l_1%q_3" STRING NOT NULL AS (CASE WHEN col1_2 IS NULL THEN e'\x17i':::STRING ELSE e'\x11r\x0e\x05\x14ZuS':::STRING END) VIRTUAL, FAMILY (_col1_0, col1_2), FAMILY ("co,l1_1"));
    rsg_test.go:683: CREATE TABLE "tabl%ve2" ("col2_*0" STRING NULL, col2_1 STRING COLLATE da_DK NULL, col2_2 REGTYPE, "col2_\\U000E5C103" "char" NOT NULL, " col2_4" STRING COLLATE en_US NOT NULL, "Co😳l2_5" REGTYPE, "|col2_6" FLOAT4, "Col2_7" CHAR NOT NULL, col2_8 GEOMETRY NOT NULL, col2_9 INTERVAL, col2_10 TSQUERY NULL, col2_11 DECIMAL NOT NULL, "%col2_12" CHAR NULL, col2_13 INT8 NOT NULL, col2_14 CHAR NOT NULL, col2_15 BYTES NOT NULL, col2_16 TSVECTOR NOT NULL, FAMILY (col2_16), FAMILY (" col2_4", "%col2_12", col2_15), FAMILY (col2_13, "Co😳l2_5", "col2_*0"), FAMILY ("|col2_6"), FAMILY (col2_1), FAMILY ("Col2_7", col2_9, col2_2), FAMILY (col2_11, col2_8), FAMILY ("col2_\\U000E5C103"), FAMILY (col2_14), FAMILY (col2_10));
    rsg_test.go:683: CREATE TABLE "tabl%qe3" (col3_0 CHAR NOT NULL, c̤ol3_1 DATE, col3_2 JSONB NOT NULL, col3_3 CHAR NOT NULL, "c
ol3_4" INT4 NOT NULL, col3_5 DATE NOT NULL, "col3%p_6" STRING NOT NULL AS (lower(col3_0)) STORED, "co%ql3_7" STRING NOT NULL AS (lower(CAST(col3_2 AS STRING))) STORED, "co!l3_8" STRING AS (CASE WHEN c̤ol3_1 IS NULL THEN 'X':::STRING ELSE 'B)>':::STRING END) VIRTUAL, PRIMARY KEY (col3_2 ASC, "col3%p_6", "c
ol3_4" DESC, col3_3 ASC, col3_0 ASC), INVERTED INDEX (col3_2, col3_0 ASC, c̤ol3_1, "col3%p_6" DESC, "co!l3_8" ASC, col3_5 DESC, "co%ql3_7" gin_trgm_ops), UNIQUE (col3_2 ASC, "co%ql3_7" ASC, col3_0, "col3%p_6" ASC, (CASE WHEN c̤ol3_1 IS NULL THEN e'\'H\'G\x01H4A':::STRING ELSE '3U[aC W':::STRING END) ASC), FAMILY (col3_5), FAMILY ("co%ql3_7"), FAMILY ("c
ol3_4", col3_2), FAMILY (c̤ol3_1), FAMILY (col3_3), FAMILY (col3_0, "col3%p_6"));
    rsg_test.go:683: ALTER TABLE table1 INJECT STATISTICS e'[{"avg_size": 1, "columns": ["\\"co l_1%q_3\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4133536711555265705, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6182010907706319852}, {"avg_size": 3, "columns": ["_col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5434130813497147152, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6182010907706319852}, {"avg_size": 1, "columns": ["\\"co,l1_1\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1577919990817867707, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6182010907706319852}, {"avg_size": 8, "columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5199341348994791923, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6182010907706319852}]':::JSONB;
    rsg_test.go:683: ALTER TABLE "tabl%ve2" INJECT STATISTICS e'[{"avg_size": 16, "columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2748516993322173702, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3262440459761300284}, {"avg_size": 19, "columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2110843960102457133, "histo_col_type": "", "name": "__auto__", "null_count": 1797010920933362550, "row_count": 3262440459761300284}, {"avg_size": 30, "columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2196594189059810178, "histo_col_type": "", "name": "__auto__", "null_count": 216119999206801983, "row_count": 3262440459761300284}, {"avg_size": 7, "columns": ["\\" col2_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 332827531024969657, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3262440459761300284}, {"avg_size": 19, "columns": ["\\"|col2_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 131099914878910345, "histo_col_type": "", "name": "__auto__", "null_count": 2671507361650031337, "row_count": 3262440459761300284}, {"avg_size": 6, "columns": ["\\"col2_\\\\\\\\U000E5C103\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3128956569613334257, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3262440459761300284}, {"avg_size": 5, "columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 598764300585240726, "histo_col_type": "", "name": "__auto__", "null_count": 3090921944244231611, "row_count": 3262440459761300284}, {"avg_size": 5, "columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2730734742613917439, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3262440459761300284}, {"avg_size": 9, "columns": ["\\"%col2_12\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 496885478570431877, "histo_col_type": "", "name": "__auto__", "null_count": 2389735533008160557, "row_count": 3262440459761300284}, {"avg_size": 14, "columns": ["\\"Col2_7\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2717783360796468328, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3262440459761300284}, {"avg_size": 29, "columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1455461823722155242, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3262440459761300284}, {"avg_size": 11, "columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2909385129629930761, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3262440459761300284}, {"avg_size": 27, "columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2393436570289879828, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3262440459761300284}, {"avg_size": 17, "columns": ["\\"col2_*0\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1271485602295772821, "histo_col_type": "", "name": "__auto__", "null_count": 2409025778899145212, "row_count": 3262440459761300284}, {"avg_size": 12, "columns": ["\\"Co\U0001F633l2_5\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 427293134598520532, "histo_col_type": "", "name": "__auto__", "null_count": 3238747271478954353, "row_count": 3262440459761300284}, {"avg_size": 5, "columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2042840849599552110, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3262440459761300284}, {"avg_size": 6, "columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3032414045803194278, "histo_col_type": "", "name": "__auto__", "null_count": 1920973785533611970, "row_count": 3262440459761300284}]':::JSONB;
    rsg_test.go:683: ALTER TABLE "tabl%qe3" INJECT STATISTICS e'[{"avg_size": 4, "columns": ["\\"col3%p_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 613183766, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 18, "columns": ["\\"co%ql3_7\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 538979433, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 31, "columns": ["\\"co!l3_8\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 336840514, "histo_col_type": "", "name": "__auto__", "null_count": 558866425, "row_count": 700000000}, {"avg_size": 0, "columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 409957171, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 3, "columns": ["c\u0324ol3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 416816584, "histo_col_type": "", "name": "__auto__", "null_count": 364625593, "row_count": 700000000}, {"avg_size": 6, "columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 177931450, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 14, "columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 576031804, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "[\\"KxJ7GyZt\\"]"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4317567020318176806, "upper_bound": "[{\\"c\\": false, \\"piKLxObtP\\": true}, 3.6018524526502826]"}, {"distinct_range": 4668958452702540000, "num_eq": 805894816800127127, "num_range": 4668958452702539314, "upper_bound": "[\\"Yu\\", \\"a\\", 0.00614100537655464]"}, {"distinct_range": 7239679385.215567, "num_eq": 9000000000, "num_range": 8000000000, "upper_bound": "[false, {\\"c\\": null}, 0.7404590462287186]"}, {"distinct_range": 4978494402176085000, "num_eq": 200000, "num_range": 7720433737784918936, "upper_bound": "[{\\"a\\": false, \\"foo\\": null, \\"rJ4wUyJ\\": 0.13610275622560025}, [[{\\"hmuaX\\": false}, false, \\"8SmOl\\", false], \\"veWri3M2H\\", null], {\\"0pl\\": false, \\"a\\": true}, 0.30221510519499617]"}, {"distinct_range": 944711.0252936108, "num_eq": 1473472462804871813, "num_range": 1000000, "upper_bound": "{}"}, {"distinct_range": 0, "num_eq": 6877176870264841128, "num_range": 4403596187664572311, "upper_bound": "{\\"test\\": \\"json\\"}"}], "histo_col_type": "JSONB", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 22, "columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 434724647, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 22, "columns": ["\\"c\\rol3_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 662477759, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}]':::JSONB;
    rsg_test.go:683: CREATE TYPE rand_typ_0 AS ENUM ('ac', 'dat');
    rsg_test.go:683: CREATE TYPE rand_typ_1 AS ENUM ('o', 'nl', 'f', 'obq', 'ua');
    rsg_test.go:683: SET sql_safe_updates = false;;
    rsg_test.go:683: 
        BEGIN; CREATE TYPE greeting AS ENUM ('hello', 'howdy', 'hi', 'good day', 'morning'); COMMIT;
        BEGIN;
        CREATE TABLE IF NOT EXISTS seed AS
        	SELECT
        		g::INT2 AS _int2,
        		g::INT4 AS _int4,
        		g::INT8 AS _int8,
        		g::FLOAT4 AS _float4,
        		g::FLOAT8 AS _float8,
        		'2001-01-01'::DATE + g AS _date,
        		'2001-01-01'::TIMESTAMP + g * '1 day'::INTERVAL AS _timestamp,
        		'2001-01-01'::TIMESTAMPTZ + g * '1 day'::INTERVAL AS _timestamptz,
        		g * '1 day'::INTERVAL AS _interval,
        		g % 2 = 1 AS _bool,
        		g::DECIMAL AS _decimal,
        		g::STRING AS _string,
        		g::STRING::BYTES AS _bytes,
        		substring('00000000-0000-0000-0000-' || g::STRING || '00000000000', 1, 36)::UUID AS _uuid,
        		'0.0.0.0'::INET + g AS _inet,
        		g::STRING::JSONB AS _jsonb,
        		enum_range('hello'::greeting)[g] as _enum
        	FROM
        		generate_series(1, 5) AS g;
        COMMIT;
        
        INSERT INTO seed DEFAULT VALUES;
        CREATE INDEX on seed (_int8, _float8, _date);
        CREATE INVERTED INDEX on seed (_jsonb);
        ;
    rsg_test.go:685: 
    rsg_test.go:686: -- test log scope end --
--- FAIL: TestRandomSyntaxSQLSmith (303.20s)
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxSQLSmith2410092763
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSQLSmith.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-30687

sql/tests.TestRandomSyntaxSQLSmith [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11373143?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11373143?buildTab=artifacts#/) on master @ [efa3cfa5574d67640ad58d8437a259edd61dbf40](https://github.com/cockroachdb/cockroach/commits/efa3cfa5574d67640ad58d8437a259edd61dbf40):

Random syntax error:

```
    rsg_test.go:878: Crash detected: server panic: statement exec timeout
```
Query:

```
        CREATE FUNCTION "\\u6F23func135"(IN p0 TIME, IN p1 rand_typ_3, IN p2 STRING, IN p3 TSVECTOR, IN p4 REGPROCEDURE)
        	RETURNS greeting
        	LANGUAGE SQL
        	AS $funcbody$SELECT ('17 years 9 mons 598 days 05:34:15.435255':::INTERVAL::INTERVAL + tab3610._date::DATE)::TIMESTAMP AS col8475, tab3610._float4 AS col8476, p3 AS col8477, p1 AS c🙁ol8478, tab3610.tableoid AS col8479, (-0.7233419520206952):::FLOAT8 AS col😺8480, tab3610._date AS col8481, p4 AS col8482, (-373756.7540384502308):::DECIMAL AS "?co\nl8483", 768136880278784563:::INT8 AS "c
ol8484", tab3610._uuid AS " col8485", tab3610._bytes AS "col 8486", e'\x0e\x12\x1e':::STRING AS col8487, p2 AS col8488, tab3610._int4 AS """col😁8489", tab3610._jsonb AS "c%vol8490" FROM defaultdb.public.seed@[0] AS tab3610 WHERE tab3610._bool;
```
Schema:

```
    rsg_test.go:688: To reproduce, use schema:
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;;
    rsg_test.go:690: CREATE TABLE table😈1 (col1_0 INT2 NOT NULL, col1_1 REGCLASS NOT NULL, c😂ol1_2 JSONB NOT NULL, """col1_3" JSONB NOT NULL, col1_4 INT4 NOT NULL, col1_5 INT8 NOT NULL AS (col1_0 + col1_4) STORED, col1_6 INT8 NOT NULL AS (col1_0 + col1_4) STORED, col1_7 STRING NOT NULL AS (CASE WHEN col1_1 IS NULL THEN 'Lt;^':::STRING ELSE NULL END) VIRTUAL, PRIMARY KEY (col1_1 DESC, col1_6, col1_5 DESC, col1_7 DESC, col1_0, col1_4, c😂ol1_2 DESC), UNIQUE ("""col1_3" DESC, col1_6 DESC, col1_5, col1_7, col1_0 ASC, col1_1 DESC) WHERE ((((table😈1.col1_7 >= e'\x00':::STRING) OR (table😈1.col1_4 != (-32768):::INT8)) OR (table😈1.col1_5 >= 32767:::INT8)) AND (table😈1.col1_6 <= 9223372036854775807:::INT8)) AND (table😈1.col1_0 <= (-1):::INT8), INDEX (col1_5, col1_0 ASC, col1_6 DESC, col1_4 ASC, col1_1 DESC, """col1_3" ASC, (CASE WHEN col1_1 IS NULL THEN 'FZO8#':::STRING ELSE e'\x00':::STRING END) ASC) WHERE ((((table😈1.col1_4 >= (-32768):::INT8) OR (table😈1.col1_5 != (-32768):::INT8)) AND (table😈1.col1_7 > '':::STRING)) OR (table😈1.col1_6 <= 2147483647:::INT8)) OR (table😈1.col1_0 = 127:::INT8), UNIQUE ((col1_0 + 2264:::INT8) DESC, col1_5, lower(CAST(c😂ol1_2 AS STRING)) DESC, col1_1, c😂ol1_2, """col1_3" DESC, col1_7 DESC), INDEX (c😂ol1_2) PARTITION BY LIST (c😂ol1_2) (PARTITION "table😈1_par
        t0" VALUES IN (('{"JK": "foobar", "bar": [true], "foo": 0.3686539545670234, "ywtET6": [true, true, false, true]}':::JSONB,), ('{"VVPwK3bBX": {"ce3nWR": 3.6953729974916945}, "WvRMbzpWn": "CzK2caL", "a": true}':::JSONB,), ('{"G5": false, "a": [{"b": null, "foobar": 0.597809375709615}, "4S8wC3Q", {"jQr": 0.7689037351967372}, "foo"], "foobar": 0.036973023167798796, "yQ6": false}':::JSONB,), ('{"b": {"b": "bar", "bar": 0.6240992888237471, "rQUx": "yxBCxn"}, "dybPqt": null, "foobar": null, "hO": true}':::JSONB,), ('{"ERdruqqG1": null, "a": 1.149289388771084, "foo": false, "iwsu": [{"37RKU": true, "a": null}, 2.195555541300064]}':::JSONB,), ('[{"foo": false, "foobar": false}, {"1DDo": true, "DERJ": false, "NW3b": ["foobar"]}, [0.9059805621431085], false]':::JSONB,))) WHERE ((((table😈1.col1_0 = 1:::INT8) OR (table😈1.col1_7 = 'X':::STRING)) OR (table😈1.col1_4 = 0:::INT8)) OR (table😈1.col1_5 >= 127:::INT8)) OR (table😈1.col1_6 <= (-9223372036854775807):::INT8), INDEX (col1_7), UNIQUE ((col1_4 + col1_0) ASC) STORING ("""col1_3") WHERE ((((table😈1.col1_5 != 1:::INT8) OR (table😈1.col1_7 = '':::STRING)) AND (table😈1.col1_4 >= (-1):::INT8)) OR (table😈1.col1_6 > 32767:::INT8)) AND (table😈1.col1_0 > (-1):::INT8), FAMILY ("""col1_3", c😂ol1_2, col1_4, col1_5, col1_0, col1_6), FAMILY (col1_1));
    rsg_test.go:690: CREATE TABLE "tab/le2" (col2😬_0 JSONB NOT NULL, col2_1 TIME NOT NULL, "col""2_2" TIMETZ NOT NULL, col2_͌3 STRING NOT NULL AS (lower(CAST("col""2_2" AS STRING))) VIRTUAL, PRIMARY KEY (col2_1 ASC, "col""2_2" DESC));
    rsg_test.go:690: CREATE TABLE "%ptable3" (col̞3_0 STRING NOT NULL, "col'3_1" CHAR, PRIMARY KEY (col̞3_0), UNIQUE ("col'3_1", col̞3_0 DESC), UNIQUE ("col'3_1" DESC, col̞3_0 DESC) WHERE ("%ptable3"."col'3_1" > '':::STRING) OR ("%ptable3".col̞3_0 > '':::STRING), INDEX ("col'3_1", col̞3_0) WHERE ("%ptable3"."col'3_1" <= e'\'':::STRING) OR ("%ptable3".col̞3_0 != 'X':::STRING), INVERTED INDEX (col̞3_0 DESC, "col'3_1" gin_trgm_ops) WHERE ("%ptable3".col̞3_0 < '':::STRING) AND ("%ptable3"."col'3_1" >= e'\U00002603':::STRING), UNIQUE ("col'3_1" ASC, col̞3_0 DESC) WHERE ("%ptable3".col̞3_0 < 'X':::STRING) AND ("%ptable3"."col'3_1" >= e'\x00':::STRING), INDEX ("col'3_1" DESC, col̞3_0 ASC) PARTITION BY LIST ("col'3_1") (PARTITION "%ptable3_part0" VALUES IN (('v':::STRING,), ('3':::STRING,), (' ':::STRING,), ('r':::STRING,), ('8':::STRING,), ('9':::STRING,), ('':::STRING,)), PARTITION " %ptable3_%8bpart1" VALUES IN ((e'\x00':::STRING,), ('y':::STRING,), (']':::STRING,), (e'\x1f':::STRING,), (e'\t':::STRING,), ('*':::STRING,)), PARTITION "%ptable3_part2" VALUES IN (('U':::STRING,), ('"':::STRING,), ('W':::STRING,), ('e':::STRING,), ('g':::STRING,), ('i':::STRING,)), PARTITION "DEFAULT" VALUES IN ((DEFAULT,))) VISIBILITY 0.39);
    rsg_test.go:690: ALTER TABLE table😈1 INJECT STATISTICS e'[{"avg_size": 15, "columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3175452473038065656, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4728382748067373639}, {"avg_size": 26, "columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 654345384370226521, "histo_buckets": [{"distinct_range": 0, "num_eq": 8277619605157786799, "num_range": 0, "upper_bound": "2957319763"}, {"distinct_range": 3442485090585183700, "num_eq": 900000, "num_range": 3442485090585183731, "upper_bound": "3256004403"}, {"distinct_range": 8130827833231590000, "num_eq": 90000000, "num_range": 8130827833231590028, "upper_bound": "4005128602"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 4728382748067373639}, {"avg_size": 23, "columns": ["c\U0001F602ol1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3385285689760057910, "histo_buckets": [{"distinct_range": 0, "num_eq": 2410350244476858056, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 1634733367880517790, "num_range": 1090846118752562937, "upper_bound": "[0.7483242451533292, false, [null, null, 0.6822233140528647], {\\"7d7zbEAW\\": \\"bar\\"}, 0.41788110275596896, 0.6563117624464414]"}, {"distinct_range": 33629898.25964024, "num_eq": 0, "num_range": 50000000, "upper_bound": "{}"}, {"distinct_range": 0, "num_eq": 2098875859393434665, "num_range": 5000, "upper_bound": "{\\"test\\": \\"json\\"}"}, {"distinct_range": 4776234771250201000, "num_eq": 7656891887285007393, "num_range": 4776234771250200419, "upper_bound": "{\\"ey\\": 0.37041510283791723, \\"jcosfcBf\\": false}"}, {"distinct_range": 2818656334145442000, "num_eq": 400000, "num_range": 2824534465627019253, "upper_bound": "{\\"BMDryZxIG\\": null, \\"Dn49J0AM\\": true, \\"ElJJaE9sL\\": [[true], \\"b\\", [\\"bar\\"], 0.7683300154946618, true], \\"VA7fTgqN\\": true}"}, {"distinct_range": 236713121963289540, "num_eq": 2263533464546687740, "num_range": 376491115582981142, "upper_bound": "{\\"273c\\": null, \\"b\\": \\"a\\", \\"dK9mBVrFR\\": null, \\"fz\\": true, \\"qod\\": {\\"ah\\": null, \\"b\\": {\\"gIshb\\": 7.125588145971881, \\"ooNxeyFj\\": [true]}, \\"c\\": 1.907005632965552}}"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 60000000000, "upper_bound": "{\\"8GIcd\\": {\\"foo\\": \\"oD\\", \\"h2WdX\\": {\\"8Q7h9mTmg\\": \\"avjjZm88p\\", \\"xCsoIxAz\\": true}, \\"zJbNm4FZ\\": [null]}, \\"a\\": [0.07758278237775684, 2.1754386255447136, true], \\"baz\\": \\"a\\", \\"foo\\": 1.020884294754539, \\"tK1ftY\\": [true]}"}], "histo_col_type": "JSONB", "name": "__auto__", "null_count": 0, "row_count": 4728382748067373639}, {"avg_size": 28, "columns": ["\\"\\"\\"col1_3\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1868652951934721335, "histo_buckets": [{"distinct_range": 0, "num_eq": 4796808371039140279, "num_range": 0, "upper_bound": "[0.14505392801941705]"}, {"distinct_range": 0, "num_eq": 111284834740474458, "num_range": 30000000000, "upper_bound": "[false]"}, {"distinct_range": 5660570037473099000, "num_eq": 10000000000, "num_range": 5660570037473098743, "upper_bound": "[[true, true], 2.546009469767416]"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 70, "upper_bound": "[{\\"3stCM\\": true, \\"a\\": [true, false, true], \\"b\\": false, \\"foo\\": 0.930832769460932}, 0.5294377148397652, 0.7782954683567422]"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 1000000, "upper_bound": "[[\\"2eNqqR\\", [false, false], true, true], {\\"foo\\": 0.7028171373648069}, \\"2AEC2T\\", true, false]"}, {"distinct_range": 0, "num_eq": 4098711205236074545, "num_range": 0, "upper_bound": "[[{\\"6bcps\\": false, \\"Mz3tLGQ4\\": [null, null], \\"c\\": {\\"foobar\\": true, \\"gpZUk\\": 1.0964579547073012}}, {\\"CH4Hl9Xk\\": \\"c\\", \\"InuS\\": [[\\"bar\\"], true], \\"c\\": true, \\"ly\\": null}, [true], [0.093174306699986], {\\"aVquu\\": false}, true, \\"SdEPq\\"], {\\"YepN2tK3\\": true, \\"a\\": [null, false, true], \\"b\\": {\\"bar\\": \\"foobar\\"}, \\"bar\\": 0.6952893817329082}, [\\"baz\\", null], {\\"bfLrGIvc\\": \\"b\\", \\"sUX1e\\": null}, true, \\"a\\"]"}, {"distinct_range": 2719958361717585000, "num_eq": 6096259794292146221, "num_range": 2719958361717584781, "upper_bound": "{\\"VZAuRD\\": 1.0908647658360626, \\"foo\\": {\\"a\\": \\"foo\\"}}"}, {"distinct_range": 600000, "num_eq": 30000, "num_range": 600000, "upper_bound": "{\\"Y8\\": \\"c\\", \\"a\\": 2.3024279860551413, \\"bar\\": 1.3568868580831457}"}], "histo_col_type": "JSONB", "name": "__auto__", "null_count": 0, "row_count": 4728382748067373639}, {"avg_size": 8, "columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1499171297750161056, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4728382748067373639}, {"avg_size": 15, "columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1271139012550929714, "histo_buckets": [{"distinct_range": 0, "num_eq": 50, "num_range": 0, "upper_bound": "-6653059211195107430"}, {"distinct_range": 58.940507985864656, "num_eq": 0, "num_range": 100, "upper_bound": "-2366262493072678252"}, {"distinct_range": 7987.196875272467, "num_eq": 300000000, "num_range": 8000, "upper_bound": "1783947787957693554"}, {"distinct_range": 3907714959108644400, "num_eq": 900000000, "num_range": 3907714959108644449, "upper_bound": "6126720104868559169"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 4728382748067373639}, {"avg_size": 0, "columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1471009556640549176, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4728382748067373639}, {"avg_size": 25, "columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1204933721454195748, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4728382748067373639}]':::JSONB;
    rsg_test.go:690: ALTER TABLE "tab/le2" INJECT STATISTICS e'[{"avg_size": 3, "columns": ["col2\U0001F62C_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 705612121187338036, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4999619281300691084}, {"avg_size": 13, "columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1318728019887476293, "histo_buckets": [{"distinct_range": 0, "num_eq": 3354499230828599, "num_range": 0, "upper_bound": "00:49:27.514024"}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 3104408960617730727, "upper_bound": "04:02:52.01297"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 7555800802274751888, "upper_bound": "11:28:18.312879"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 70000000, "upper_bound": "11:53:26.407001"}, {"distinct_range": 185860.22179727512, "num_eq": 0, "num_range": 200000, "upper_bound": "12:48:59.53792"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 40, "upper_bound": "15:58:13.372433"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 20, "upper_bound": "16:34:26.114367"}, {"distinct_range": 1389613123979991300, "num_eq": 60000, "num_range": 2433779507651112408, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 4999619281300691084}, {"avg_size": 21, "columns": ["\\"col\\"\\"2_2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 805945600047117558, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4999619281300691084}, {"avg_size": 6, "columns": ["col2_\u034C3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4209389415163532855, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4999619281300691084}]':::JSONB;
    rsg_test.go:690: ALTER TABLE "%ptable3" INJECT STATISTICS e'[{"avg_size": 30, "columns": ["col\u031E3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8644176648, "histo_buckets": [{"distinct_range": 0, "num_eq": 8455133692198528752, "num_range": 0, "upper_bound": "\\u0015MFU"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 8188407977517694222, "upper_bound": "\\u0018\\u0015~,i\\u0004"}, {"distinct_range": 35820857987.88262, "num_eq": 70000, "num_range": 90000000000, "upper_bound": "7#|"}, {"distinct_range": 11506.14933050891, "num_eq": 40000000, "num_range": 30000, "upper_bound": "B,^Q\\u001epx"}, {"distinct_range": 3841065393926049000, "num_eq": 3317916243365069877, "num_range": 3841065393926048908, "upper_bound": "[j0g!"}, {"distinct_range": 200, "num_eq": 7540735946011493351, "num_range": 200, "upper_bound": "m}\\u001de\\txe\\u0003"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"avg_size": 11, "columns": ["\\"col\'3_1\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5938520007, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 4000000, "num_eq": 357611707243734719, "num_range": 4000000, "upper_bound": "\\r"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 60000000000, "upper_bound": "\\u000e"}, {"distinct_range": 1000, "num_eq": 8418869264883176323, "num_range": 1000, "upper_bound": "\\u0011"}, {"distinct_range": 1652970443695784200, "num_eq": 2000000000, "num_range": 1652970443695784237, "upper_bound": " "}, {"distinct_range": 8538160905995180000, "num_eq": 2006758792761837552, "num_range": 8538160905995180120, "upper_bound": "/"}, {"distinct_range": 8368619130410732000, "num_eq": 50, "num_range": 8368619130410731755, "upper_bound": "G"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 100000000000, "upper_bound": "Q"}, {"distinct_range": 1935194513441026000, "num_eq": 5888891044304982655, "num_range": 1935194513441026055, "upper_bound": "x"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 6919975652, "row_count": 9000000000}]':::JSONB;
    rsg_test.go:690: CREATE TYPE rand_typ_0 AS ENUM ('k', 'blqkqj', 'rt', 'tbaug', 'x', 'mjebk');
    rsg_test.go:690: CREATE TYPE rand_typ_1 AS ENUM ('owk', 'hm', 'vmgj', 'ffubu');
    rsg_test.go:690: CREATE TYPE rand_typ_2 AS ENUM ('o', 'rxocqs');
    rsg_test.go:690: CREATE TYPE rand_typ_3 AS ENUM ('rgltt', 'ol', 'ldkzw');
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: 
        BEGIN; CREATE TYPE greeting AS ENUM ('hello', 'howdy', 'hi', 'good day', 'morning'); COMMIT;
        BEGIN;
        CREATE TABLE IF NOT EXISTS seed AS
        	SELECT
        		g::INT2 AS _int2,
        		g::INT4 AS _int4,
        		g::INT8 AS _int8,
        		g::FLOAT4 AS _float4,
        		g::FLOAT8 AS _float8,
        		'2001-01-01'::DATE + g AS _date,
        		'2001-01-01'::TIMESTAMP + g * '1 day'::INTERVAL AS _timestamp,
        		'2001-01-01'::TIMESTAMPTZ + g * '1 day'::INTERVAL AS _timestamptz,
        		g * '1 day'::INTERVAL AS _interval,
        		g % 2 = 1 AS _bool,
        		g::DECIMAL AS _decimal,
        		g::STRING AS _string,
        		g::STRING::BYTES AS _bytes,
        		substring('00000000-0000-0000-0000-' || g::STRING || '00000000000', 1, 36)::UUID AS _uuid,
        		'0.0.0.0'::INET + g AS _inet,
        		g::STRING::JSONB AS _jsonb,
        		enum_range('hello'::greeting)[g] as _enum
        	FROM
        		generate_series(1, 5) AS g;
        COMMIT;
        
        INSERT INTO seed DEFAULT VALUES;
        CREATE INDEX on seed (_int8, _float8, _date);
        CREATE INVERTED INDEX on seed (_jsonb);
        ;
    rsg_test.go:692: 
    rsg_test.go:693: -- test log scope end --
--- FAIL: TestRandomSyntaxSQLSmith (304.01s)
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxSQLSmith2962649716
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSQLSmith.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSQLSmith [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11390665?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11390665?buildTab=artifacts#/) on master @ [9fcc5c8ce60d2f25e06e41bd97e20445233f9fa5](https://github.com/cockroachdb/cockroach/commits/9fcc5c8ce60d2f25e06e41bd97e20445233f9fa5):

Random syntax error:

```
    rsg_test.go:880: Crash detected: server panic: statement exec timeout
```
Query:

```
        CREATE FUNCTION func289(IN p0 INT2, IN p1 VARCHAR[], IN p2 BIT(45), IN p3 "char", IN p4 REGTYPE, IN p5 REGCLASS, IN p6 INET)
        	RETURNS RECORD
        	NOT LEAKPROOF
        	LANGUAGE SQL
        	AS $funcbody$SELECT B'11011110111111111000010011010' AS col20354, t😗ab7790."col3_%p9" AS col20355, t😗ab7790.col3_5 AS "co�l20356", t😗ab7790."col'3_4" AS "%5ecol20357", p2 AS "c|ol20358", '12e6:2a40:52a5:6431:358b:3b95:6599:6135/110':::INET AS "😑cOl20359", '?RV':::STRING AS colͅ20360, t😗ab7790."col3_%p9" AS "c\\x7fol20361", p4 AS col̑20362 FROM defaultdb.public."tabLe3"@"tabLe3_pkey" AS t😗ab7790 LIMIT 57:::INT8;
```
Schema:

```
    rsg_test.go:688: To reproduce, use schema:
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;;
    rsg_test.go:690: CREATE TABLE table1 (col1_0 JSONB NOT NULL, "C
ol1_1" OID NULL, col1_2 TIME NOT NULL, col1_3 GEOGRAPHY NOT NULL, "ć%qol1_4" REGTYPE NOT NULL, "'cől1__5" "char" NOT NULL, "col1�_6" TIME NULL, col1_7 BIT(19) NOT NULL, "'col1_8" STRING NOT NULL, "col1_ 9" INT4 NULL, PRIMARY KEY ("ć%qol1_4" DESC, col1_7), INDEX (col1_2, "'cől1__5" DESC, lower(CAST(col1_3 AS STRING)) ASC, "'col1_8" ASC, "ć%qol1_4" DESC) WHERE ((((table1."col1�_6" = '24:00:00':::TIME) AND (table1.col1_2 >= '24:00:00':::TIME)) OR (table1."'col1_8" != e'\x00':::STRING)) AND (table1."'cől1__5" > e'\x00':::STRING)) AND (table1."col1_ 9" < 127:::INT8), UNIQUE ("'col1_8" DESC, col1_0 ASC, (CASE WHEN "ć%qol1_4" IS NULL THEN e'S(\tFyjP':::STRING ELSE e'\x1e:':::STRING END) DESC, lower(CAST(col1_7 AS STRING)), "ć%qol1_4") WHERE (((table1."col1�_6" < '24:00:00':::TIME) OR (table1.col1_2 = '24:00:00':::TIME)) OR (table1."col1_ 9" >= (-128):::INT8)) AND (table1."'cől1__5" != '':::STRING), UNIQUE ("'col1_8", col1_0 ASC, "'cől1__5" DESC, col1_2 DESC, "col1_ 9" DESC, "C
ol1_1") STORING (col1_3, "col1�_6"), UNIQUE ("C
ol1_1" DESC, "col1_ 9", col1_7, lower(CAST("col1�_6" AS STRING)) ASC) STORING (col1_0, col1_2, col1_3, "'cől1__5", "'col1_8") WHERE ((table1."'col1_8" <= e'\x00':::STRING) OR (table1."'cől1__5" != e'\U00002603':::STRING)) AND (table1."col1_ 9" <= 127:::INT8));
    rsg_test.go:690: CREATE TABLE tableͨ2 (col2_0 GEOGRAPHY, col2_1 UUID NULL, "col'2_2" TIMESTAMPTZ NOT NULL, col2_3 NAME[] NOT NULL, c̪ol2_4 INET NULL, col2_5 GEOGRAPHY NULL, col2_6 OID NULL, col2_😜7 STRING NULL AS (lower(CAST(col2_5 AS STRING))) STORED, "col2_ 8" STRING NOT NULL AS (CASE WHEN "col'2_2" IS NULL THEN '':::STRING ELSE e'\x7f(D^g&i\x04':::STRING END) VIRTUAL, "{col😞2_9" STRING NULL AS (lower(CAST(col2_1 AS STRING))) STORED, INVERTED INDEX (col2_😜7 gin_trgm_ops ASC) WHERE (((tableͨ2."col'2_2" = '3000-01-01 00:00:00+00':::TIMESTAMPTZ) AND (tableͨ2."col2_ 8" <= '':::STRING)) OR (tableͨ2."{col😞2_9" > e'\x00':::STRING)) AND (tableͨ2.col2_😜7 != '':::STRING), INDEX (c̪ol2_4 ASC, "col'2_2" ASC, col2_1 DESC) WHERE tableͨ2."col'2_2" > '294276-12-31 23:59:59.999999+00':::TIMESTAMPTZ, FAMILY (col2_6, col2_0, c̪ol2_4, col2_😜7, "col'2_2"), FAMILY (col2_5, col2_1), FAMILY ("{col😞2_9", col2_3));
    rsg_test.go:690: CREATE TABLE "tabLe3" (c😚ol3_0 BOOL NOT NULL, col3_1 BOOL NOT NULL, col3_2 REGROLE NULL, "col3_!\\u518D3" BIT(29) NOT NULL, "col'3_4" STRING COLLATE da_DK NOT NULL, col3_5 UUID NULL, col3_6 "char" NOT NULL, 😻col3_7 STRING NOT NULL AS (lower(col3_6)) VIRTUAL, "col3_ 8" STRING NULL AS (lower(CAST(col3_5 AS STRING))) VIRTUAL, "col3_%p9" STRING NOT NULL AS (lower(CAST(col3_5 AS STRING))) VIRTUAL, "co|l3_10" STRING NOT NULL AS (lower(CAST(c😚ol3_0 AS STRING))) VIRTUAL, col3_11 STRING NOT NULL AS (lower(CAST(c😚ol3_0 AS STRING))) STORED, PRIMARY KEY ("col'3_4" ASC, "col3_%p9", col3_6 ASC, "co|l3_10" ASC, col3_1 DESC, 😻col3_7 DESC), INDEX (lower(CAST(c😚ol3_0 AS STRING)) DESC, "col3_%p9", "col3_ 8" DESC, "col'3_4", lower(CAST(col3_5 AS STRING)), "co|l3_10" DESC, col3_1 ASC, 😻col3_7 ASC, col3_5 ASC, col3_11 DESC, "col3_!\\u518D3" ASC), INDEX (c😚ol3_0, col3_11 DESC, col3_6) STORING (col3_2, "col3_!\\u518D3", col3_5), UNIQUE ("col3_!\\u518D3" ASC, c😚ol3_0, col3_2 ASC, "col3_%p9" DESC, "col'3_4") WHERE (((((("tabLe3"."col3_ 8" < e'\U00002603':::STRING) OR ("tabLe3"."col3_%p9" > e'\x00':::STRING)) OR "tabLe3".col3_1) AND ("tabLe3".col3_6 > e'\U00002603':::STRING)) AND ("tabLe3".😻col3_7 > e'\'':::STRING)) AND ("tabLe3".col3_11 <= '"':::STRING)) AND (NOT "tabLe3".c😚ol3_0), UNIQUE (col3_5 DESC, col3_1 DESC, 😻col3_7, "col3_ 8" ASC, "co|l3_10", lower(CAST(col3_5 AS STRING)) DESC, "col'3_4" ASC) STORING (c😚ol3_0, col3_11), INDEX ("col3_!\\u518D3" DESC, col3_11, "col3_ 8" ASC, col3_1 ASC, "col'3_4", "col3_%p9" DESC, col3_2 ASC, col3_5 ASC) STORING (c😚ol3_0), INDEX (col3_11 ASC, "co|l3_10" ASC, col3_2 ASC, col3_6, 😻col3_7, "col3_!\\u518D3" DESC, "col3_ 8" ASC, col3_1 ASC, "col3_%p9" DESC, lower(CAST("col'3_4" AS STRING)), "col'3_4" ASC) WHERE (((((((NOT "tabLe3".c😚ol3_0) AND ("tabLe3"."co|l3_10" <= e'\'':::STRING)) OR ("tabLe3".col3_6 >= '':::STRING)) AND ("tabLe3".col3_11 != e'\x00':::STRING)) AND ("tabLe3".😻col3_7 != 'X':::STRING)) OR "tabLe3".col3_1) OR ("tabLe3"."col3_ 8" = 'X':::STRING)) OR ("tabLe3"."col3_%p9" < '"':::STRING) NOT VISIBLE, FAMILY (col3_11), FAMILY (col3_1), FAMILY (col3_2, "col'3_4", col3_6), FAMILY ("col3_!\\u518D3"), FAMILY (col3_5, c😚ol3_0));
    rsg_test.go:690: CREATE TABLE table4 (col4_0 BIT(49) NOT NULL, "col""4_1" BIT(28) NOT NULL, col4_2 TIMESTAMPTZ, "col4/_3" TIMETZ NOT NULL, "col4|_4" TIMETZ, col4_5 BIT(19) NOT NULL, "c�ol4_6" OID, "co�l4_7" STRING[] NOT NULL, "\\u66D4c\\U0007D505ol4_8" FLOAT4 NOT NULL, col4_9 GEOMETRY NOT NULL, col͗4_10 STRING NOT NULL AS (lower(CAST("col4/_3" AS STRING))) STORED, col4_11 STRING NOT NULL AS (CASE WHEN "co�l4_7" IS NULL THEN e'tDuH\x0e':::STRING ELSE e'rSm\x034pN\x13':::STRING END) STORED, "c ol4_12" STRING AS (lower(CAST("col4|_4" AS STRING))) VIRTUAL, " col4_13" FLOAT8 NOT NULL AS ("\\u66D4c\\U0007D505ol4_8" + 1.989686131477356:::FLOAT8) VIRTUAL, col4_😎14 STRING NOT NULL AS (CASE WHEN "co�l4_7" IS NULL THEN e'\x13[1[$x':::STRING ELSE 'X':::STRING END) STORED, "c%pol4_15" STRING NOT NULL AS (CASE WHEN "co�l4_7" IS NULL THEN e'"\x1b<GM\x1b':::STRING ELSE e'2(p\x17\x1d\x13':::STRING END) STORED, "co'l4_16" STRING NOT NULL AS (lower(CAST("col""4_1" AS STRING))) STORED, col4_17 STRING AS (CASE WHEN col4_2 IS NULL THEN e'=K\x18S':::STRING ELSE e'&\t':::STRING END) VIRTUAL, INDEX ("col4/_3", " col4_13" ASC, "c�ol4_6", col4_0 ASC, "\\u66D4c\\U0007D505ol4_8" ASC, (CASE WHEN "co�l4_7" IS NULL THEN '6':::STRING ELSE e'G<j{\x7f\n':::STRING END) ASC, col4_😎14 ASC, "col4|_4"), INDEX ("co�l4_7" ASC, col4_2 ASC, col4_11 DESC, "col4/_3" DESC, "col4|_4", "c ol4_12" DESC, lower(CAST("col4|_4" AS STRING)) DESC, (CASE WHEN col4_2 IS NULL THEN '':::STRING ELSE 'q':::STRING END), col4_😎14 DESC) STORING (col4_0, col4_5, "\\u66D4c\\U0007D505ol4_8", col͗4_10), FAMILY (col4_😎14), FAMILY ("\\u66D4c\\U0007D505ol4_8"), FAMILY ("col4/_3"), FAMILY (col4_11), FAMILY (col4_2, col4_5, "co'l4_16", col4_9), FAMILY (col4_0, "c�ol4_6", "col4|_4"), FAMILY (col͗4_10, "co�l4_7", "c%pol4_15"), FAMILY ("col""4_1"));
    rsg_test.go:690: CREATE TABLE table5 (col5_0 OID, col5_1 BOX2D NOT NULL, " col5_2" REGTYPE NULL, col5_3 TSQUERY NOT NULL, col5_4 UUID NOT NULL, col5_5 BOX2D[] NOT NULL, col5͞_6 STRING COLLATE da_DK NOT NULL, col5_7 STRING NOT NULL AS (lower(CAST(col5͞_6 AS STRING))) VIRTUAL, "col5*_8" STRING NOT NULL AS (lower(CAST(col5_3 AS STRING))) VIRTUAL, " col5_😇9" STRING AS (lower(CAST(col5_0 AS STRING))) VIRTUAL, col5_10 STRING NOT NULL AS (lower(CAST(col5_1 AS STRING))) STORED, col5_11 STRING NOT NULL AS (lower(CAST(col5_3 AS STRING))) STORED, INDEX (" col5_2" ASC, col5_5 ASC, col5_0, col5_11 ASC, (CASE WHEN " col5_2" IS NULL THEN e'b\t\x05VB/x%':::STRING ELSE e'G\x16XY\x10g':::STRING END), col5_10 DESC, "col5*_8" DESC, col5_1, col5͞_6 ASC, " col5_😇9"), INVERTED INDEX ("col5*_8" gin_trgm_ops ASC), INDEX (col5_10 DESC) STORING (col5_0));
    rsg_test.go:690: ALTER TABLE table1 INJECT STATISTICS e'[{"avg_size": 25, "columns": ["\\"C\\rol1_1\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16016, "histo_col_type": "", "name": "__auto__", "null_count": 14608, "row_count": 30000}, {"avg_size": 5, "columns": ["\\"\u0107%qol1_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18010, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"avg_size": 15, "columns": ["\\"\'c\u0151l1__5\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17172, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"avg_size": 16, "columns": ["\\"col1\\u000b_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24376, "histo_col_type": "", "name": "__auto__", "null_count": 23539, "row_count": 30000}, {"avg_size": 21, "columns": ["\\"col1_ 9\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8658, "histo_col_type": "", "name": "__auto__", "null_count": 23031, "row_count": 30000}, {"avg_size": 25, "columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2093, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"avg_size": 8, "columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9668, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "03:50:09.380747"}, {"distinct_range": 348455286.64184386, "num_eq": 3339965739827482391, "num_range": 400000000, "upper_bound": "04:02:48.733515"}, {"distinct_range": 0, "num_eq": 4260026567948435636, "num_range": 830631177031281673, "upper_bound": "07:38:39.601729"}, {"distinct_range": 0, "num_eq": 0, "num_range": 500000, "upper_bound": "08:36:43.543311"}, {"distinct_range": 400000, "num_eq": 10000000, "num_range": 400000, "upper_bound": "12:53:36.061784"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 1077320767017155496, "upper_bound": "13:23:46.432953"}, {"distinct_range": 0, "num_eq": 700, "num_range": 7581716755291960946, "upper_bound": "13:31:56.75436"}, {"distinct_range": 2701492769481094000, "num_eq": 1000000, "num_range": 5076312474939586714, "upper_bound": "13:55:45.700926"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"avg_size": 26, "columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20189, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"avg_size": 24, "columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20358, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"avg_size": 17, "columns": ["\\"\'col1_8\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23396, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\u0010\\u0019{"}, {"distinct_range": 4934178361328692000, "num_eq": 2543304747010458475, "num_range": 4934178361328692589, "upper_bound": "\\u001e0\\u000eC\\u001a\\u001d4z"}, {"distinct_range": 5858583677422177000, "num_eq": 3687255610511441105, "num_range": 5858583677422177744, "upper_bound": ")[WA\\u0019\\u0015m\\u0001"}, {"distinct_range": 1788986657.5093398, "num_eq": 0, "num_range": 3000000000, "upper_bound": "]%\\rn;-J5"}, {"distinct_range": 2322364884003702300, "num_eq": 4234855795794919038, "num_range": 2322364884003702044, "upper_bound": "w_&5Y;\\";"}, {"distinct_range": 10000000000, "num_eq": 0, "num_range": 10000000000, "upper_bound": "}m"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 30000}]':::JSONB;
    rsg_test.go:690: ALTER TABLE tableͨ2 INJECT STATISTICS e'[{"avg_size": 25, "columns": ["\\"col\'2_2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5937, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000}, {"avg_size": 30, "columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3089, "histo_col_type": "", "name": "__auto__", "null_count": 3149, "row_count": 9000}, {"avg_size": 30, "columns": ["col2_\U0001F61C7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1411, "histo_buckets": [{"distinct_range": 0, "num_eq": 5177749976667478044, "num_range": 0, "upper_bound": "\\u0016DX:7\\u0007"}, {"distinct_range": 169724513942626200, "num_eq": 10000, "num_range": 1472518395183922660, "upper_bound": "\\u001a6"}, {"distinct_range": 186086392.62384683, "num_eq": 109616399478015880, "num_range": 200000000, "upper_bound": "OQp]zz"}, {"distinct_range": 20000000000, "num_eq": 20, "num_range": 20000000000, "upper_bound": "\\\\Y5\\u0010#]<"}, {"distinct_range": 78321575652128600, "num_eq": 0, "num_range": 1815590283184832680, "upper_bound": "hK\\u0006"}, {"distinct_range": 100000000, "num_eq": 8000, "num_range": 100000000, "upper_bound": "xX]BY"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 8755, "row_count": 9000}, {"avg_size": 10, "columns": ["\\"{col\U0001F61E2_9\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7149, "histo_col_type": "", "name": "__auto__", "null_count": 2217, "row_count": 9000}, {"avg_size": 16, "columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2030, "histo_col_type": "", "name": "__auto__", "null_count": 3771, "row_count": 9000}, {"avg_size": 6, "columns": ["\\"col2_ 8\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1239, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000}, {"avg_size": 0, "columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1118, "histo_col_type": "", "name": "__auto__", "null_count": 3468, "row_count": 9000}, {"avg_size": 12, "columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8436, "histo_col_type": "", "name": "__auto__", "null_count": 5598, "row_count": 9000}, {"avg_size": 25, "columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8180, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000}, {"avg_size": 26, "columns": ["c\u032Aol2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3700, "histo_col_type": "", "name": "__auto__", "null_count": 6837, "row_count": 9000}]':::JSONB;
    rsg_test.go:690: ALTER TABLE "tabLe3" INJECT STATISTICS e'[{"avg_size": 8, "columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1636, "histo_col_type": "", "name": "__auto__", "null_count": 1320, "row_count": 2000}, {"avg_size": 23, "columns": ["\\"col3_!\\\\\\\\u518D3\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1466, "histo_buckets": [{"distinct_range": 0, "num_eq": 7215496835661876261, "num_range": 0, "upper_bound": "00110101011110001001101110111"}, {"distinct_range": 47473.947888591356, "num_eq": 7922238897988682169, "num_range": 60000, "upper_bound": "00111100001000011100100000001"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7493830943425421606, "upper_bound": "01111111111111111111111111111"}, {"distinct_range": 1393256.2780471996, "num_eq": 100, "num_range": 5000000, "upper_bound": "11010001100010010011111110100"}, {"distinct_range": 0, "num_eq": 7689262520553981408, "num_range": 0, "upper_bound": "11101100000001000010110101000"}], "histo_col_type": "BIT(29)", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"avg_size": 17, "columns": ["\\"col\'3_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 986, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"avg_size": 31, "columns": ["\\"col3_ 8\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 411, "histo_col_type": "", "name": "__auto__", "null_count": 262, "row_count": 2000}, {"avg_size": 2, "columns": ["\\"col3_%p9\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1791, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"avg_size": 22, "columns": ["\\"co|l3_10\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 665, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"avg_size": 6, "columns": ["c\U0001F61Aol3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1826, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"avg_size": 11, "columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 812, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"avg_size": 18, "columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1131, "histo_buckets": [{"distinct_range": 0, "num_eq": 5869586767821954573, "num_range": 0, "upper_bound": "1bd76038-265c-4d50-a052-366658ed4cdd"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 2000000, "upper_bound": "3d2bbf3e-6229-476f-9dc0-953deaa26d5a"}, {"distinct_range": 1145304295170256000, "num_eq": 7049182374381719274, "num_range": 1145304295170256058, "upper_bound": "5799a215-1c85-4b10-a21f-ac39b9bc45b7"}, {"distinct_range": 0, "num_eq": 6539686044480262853, "num_range": 80000, "upper_bound": "74a992d4-62d7-4c7b-a0cd-58c559900b77"}, {"distinct_range": 100000, "num_eq": 0, "num_range": 100000, "upper_bound": "7d3a63cb-a159-4ec1-9f46-9dd6fd215fe1"}, {"distinct_range": 8735190688867505000, "num_eq": 7432303832277228812, "num_range": 8735190688867505131, "upper_bound": "94d2fd41-4c44-42d1-ae74-9979d4ee0c52"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "d94cee76-dac2-444c-b8ca-e5e90525121a"}, {"distinct_range": 579178896.5951351, "num_eq": 8196524021683458592, "num_range": 8000000000, "upper_bound": "e5c1e310-0698-46cd-af13-959bc2d4e17a"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 394, "row_count": 2000}, {"avg_size": 7, "columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 372, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"avg_size": 25, "columns": ["\U0001F63Bcol3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 170, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"avg_size": 0, "columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 637, "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 2000}]':::JSONB;
    rsg_test.go:690: ALTER TABLE table4 INJECT STATISTICS e'[{"avg_size": 24, "columns": ["\\"c%pol4_15\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1549207952982008616, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 11, "columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 748622889100063413, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 16, "columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1277079415791363071, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 19, "columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 130512258025455443, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 16, "columns": ["\\"c ol4_12\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1056103283926501983, "histo_col_type": "", "name": "__auto__", "null_count": 94533107443245036, "row_count": 2253350119913434417}, {"avg_size": 4, "columns": ["\\" col4_13\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1151338711873787842, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 3, "columns": ["\\"col4/_3\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 745960318709340983, "histo_buckets": [{"distinct_range": 0, "num_eq": 6482116340563829549, "num_range": 0, "upper_bound": "00:00:00+15:59"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 845297010163809886, "upper_bound": "00:31:27.017193+11:34"}, {"distinct_range": 5598711997223085000, "num_eq": 3601157170716037632, "num_range": 5598711997223084561, "upper_bound": "03:07:00.302159+12:27"}, {"distinct_range": 0, "num_eq": 8311146888706713847, "num_range": 400000, "upper_bound": "06:30:20.43356+10:41"}, {"distinct_range": 60000000, "num_eq": 400000000, "num_range": 60000000, "upper_bound": "19:10:24.033634+07:53"}, {"distinct_range": 0, "num_eq": 2087967287112763692, "num_range": 3799204698776736364, "upper_bound": "09:32:14.830942-04:30"}, {"distinct_range": 30, "num_eq": 20, "num_range": 30, "upper_bound": "19:38:58.580693-03:20"}, {"distinct_range": 0, "num_eq": 1604575138836659918, "num_range": 0, "upper_bound": "13:14:26.140028-09:50"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 6, "columns": ["\\"co\\u000cl4_7\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 119871100969649544, "histo_buckets": [{"distinct_range": 0, "num_eq": 3529016687381029108, "num_range": 0, "upper_bound": "ARRAY[]"}, {"distinct_range": 421160695804221900, "num_eq": 4000000, "num_range": 1913355494038224851, "upper_bound": "ARRAY[e\'\\\\x16\\\\x1a\\\\x05}\':::STRING]"}, {"distinct_range": 0, "num_eq": 2873858954502324861, "num_range": 1648003706209524404, "upper_bound": "ARRAY[e\'\\\\\\\\&\\\\x05\':::STRING,\'P\':::STRING,e\'3\\\\bL\\\\x1e8[\':::STRING,e\'pr1>QH\\\\\\\\H5\':::STRING]"}], "histo_col_type": "STRING[]", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 26, "columns": ["col\u03574_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1782787096041402224, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 4, "columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 776387204859264564, "histo_col_type": "", "name": "__auto__", "null_count": 1944406877955738930, "row_count": 2253350119913434417}, {"avg_size": 18, "columns": ["col4_\U0001F60E14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1830764385327221458, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 13, "columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1052428203505322847, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 2, "columns": ["\\"col\\"\\"4_1\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 931300186423053136, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 26, "columns": ["\\"col4|_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 261622037544431091, "histo_col_type": "", "name": "__auto__", "null_count": 818728185504286387, "row_count": 2253350119913434417}, {"avg_size": 9, "columns": ["\\"c\\u000bol4_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 287460604755938639, "histo_col_type": "", "name": "__auto__", "null_count": 1565636874065509705, "row_count": 2253350119913434417}, {"avg_size": 26, "columns": ["\\"\\\\\\\\u66D4c\\\\\\\\U0007D505ol4_8\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1453427222493234421, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}, {"avg_size": 30, "columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 611591504291736478, "histo_col_type": "", "name": "__auto__", "null_count": 30753970979595629, "row_count": 2253350119913434417}, {"avg_size": 13, "columns": ["\\"co\'l4_16\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 937259423527213997, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2253350119913434417}]':::JSONB;
    rsg_test.go:690: ALTER TABLE table5 INJECT STATISTICS e'[{"avg_size": 29, "columns": ["\\" col5_2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 19, "row_count": 20}, {"avg_size": 24, "columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"avg_size": 1, "columns": ["col5\u035E_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"avg_size": 0, "columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 3, "row_count": 20}, {"avg_size": 27, "columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"avg_size": 16, "columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"avg_size": 12, "columns": ["\\"col5*_8\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "4w"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 20}, {"avg_size": 30, "columns": ["\\" col5_\U0001F6079\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 7, "row_count": 20}, {"avg_size": 3, "columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_buckets": [{"distinct_range": 0, "num_eq": 6424457921526732116, "num_range": 0, "upper_bound": "\\u0013KX=\\u0003"}, {"distinct_range": 0, "num_eq": 5975908732660330027, "num_range": 7198753432318840119, "upper_bound": "?y"}, {"distinct_range": 0, "num_eq": 3910290438062092768, "num_range": 897841023618637761, "upper_bound": "\\\\*\\u000cIC"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 20}, {"avg_size": 2, "columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"avg_size": 12, "columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"avg_size": 0, "columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}]':::JSONB;
    rsg_test.go:690: CREATE TYPE rand_typ_0 AS ENUM ('ru', 'fijehp', 'wzcr', 'cx', 'fh');
    rsg_test.go:690: CREATE TYPE rand_typ_1 AS ENUM ('mmrus', 'zviqsd', 'qxouv', 'aglm');
    rsg_test.go:690: CREATE TYPE rand_typ_2 AS ENUM ('c', 'dyam', 'ggqm', 'sypg');
    rsg_test.go:690: CREATE TYPE rand_typ_3 AS ENUM ('avvc', 'c', 'cq', 'tdrcqc');
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: 
        BEGIN; CREATE TYPE greeting AS ENUM ('hello', 'howdy', 'hi', 'good day', 'morning'); COMMIT;
        BEGIN;
        CREATE TABLE IF NOT EXISTS seed AS
        	SELECT
        		g::INT2 AS _int2,
        		g::INT4 AS _int4,
        		g::INT8 AS _int8,
        		g::FLOAT4 AS _float4,
        		g::FLOAT8 AS _float8,
        		'2001-01-01'::DATE + g AS _date,
        		'2001-01-01'::TIMESTAMP + g * '1 day'::INTERVAL AS _timestamp,
        		'2001-01-01'::TIMESTAMPTZ + g * '1 day'::INTERVAL AS _timestamptz,
        		g * '1 day'::INTERVAL AS _interval,
        		g % 2 = 1 AS _bool,
        		g::DECIMAL AS _decimal,
        		g::STRING AS _string,
        		g::STRING::BYTES AS _bytes,
        		substring('00000000-0000-0000-0000-' || g::STRING || '00000000000', 1, 36)::UUID AS _uuid,
        		'0.0.0.0'::INET + g AS _inet,
        		g::STRING::JSONB AS _jsonb,
        		enum_range('hello'::greeting)[g] as _enum
        	FROM
        		generate_series(1, 5) AS g;
        COMMIT;
        
        INSERT INTO seed DEFAULT VALUES;
        CREATE INDEX on seed (_int8, _float8, _date);
        CREATE INVERTED INDEX on seed (_jsonb);
        ;
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (2382067852999336474);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (5693139872233372271);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (995239976723704236);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (4179684054152447821);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (8248800360313616195);
    rsg_test.go:690: ALTER TABLE seed SCATTER;
    rsg_test.go:692: 
    rsg_test.go:693: -- test log scope end --
--- FAIL: TestRandomSyntaxSQLSmith (303.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSQLSmith.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSQLSmith [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11400807?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11400807?buildTab=artifacts#/) on master @ [31fe139c74640a383599c53719a39f0190a7065d](https://github.com/cockroachdb/cockroach/commits/31fe139c74640a383599c53719a39f0190a7065d):

Random syntax error:

```
    rsg_test.go:880: Crash detected: server panic: statement exec timeout
```
Query:

```
        CREATE FUNCTION "funC18"()
        	RETURNS SETOF OID
        	NOT LEAKPROOF
        	LANGUAGE SQL
        	AS $funcbody$SELECT "t-%qab509".tableoid AS "col%v1218", "ta %vb507"._bytes AS col1219 FROM defaultdb.public.seed@seed__int8__float8__date_idx AS "ta %vb507", defaultdb.public."table	3"@"table	3_col3\\u776E_1_key" AS tab508, defaultdb.public.table1@table1_expr_col1_2_idx AS "t-%qab509", defaultdb.public.seed@seed__int8__float8__date_idx AS tab510 LIMIT 99:::INT8;
```
Schema:

```
    rsg_test.go:688: To reproduce, use schema:
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;;
    rsg_test.go:690: CREATE TABLE table1 ("c\\xf3ol1_0" TIMESTAMPTZ NOT NULL, "col%q1_1" TSVECTOR NOT NULL, col1_2 STRING NOT NULL AS (CASE WHEN "c\\xf3ol1_0" IS NULL THEN '4U<':::STRING ELSE e'#\\\x1f{\x15\x05':::STRING END) VIRTUAL, PRIMARY KEY (col1_2, "c\\xf3ol1_0"), INVERTED INDEX ("c\\xf3ol1_0", col1_2 DESC, "col%q1_1") VISIBILITY 0.65, UNIQUE ("c\\xf3ol1_0"), INDEX (col1_2) WHERE table1."c\\xf3ol1_0" < '2001-01-01 00:00:00+00 BC':::TIMESTAMPTZ, UNIQUE ("c\\xf3ol1_0" ASC) WHERE table1."c\\xf3ol1_0" >= '3000-01-01 00:00:00+00':::TIMESTAMPTZ, INDEX (lower(CAST("col%q1_1" AS STRING)), col1_2 DESC) STORING ("col%q1_1") WHERE table1."c\\xf3ol1_0" > '4714-11-24 00:00:00+00 BC':::TIMESTAMPTZ);
    rsg_test.go:690: CREATE TABLE "t&able2" (col2_0 TIMESTAMP NOT NULL, col2_̵1 TIMESTAMP NOT NULL, col2_2 BYTES, col2_3 FLOAT4 NOT NULL, "col2 _4" TIMETZ NULL, col2_5 STRING COLLATE en_US NOT NULL, col2_6 BIT(17) NOT NULL, col2_7 TIME NOT NULL, col2_8 STRING AS (CASE WHEN col2_2 IS NULL THEN e'o\x19\x16B@O\x04':::STRING ELSE NULL END) VIRTUAL, "c ol2_9" STRING AS (CASE WHEN col2_2 IS NULL THEN NULL ELSE e'\U00002603':::STRING END) STORED, col2_10 STRING NOT NULL AS (CASE WHEN col2_0 IS NULL THEN e'cf:\x1dsD':::STRING ELSE e'k1\x0b<WB':::STRING END) VIRTUAL, col2_11 STRING NOT NULL AS (lower(CAST(col2_5 AS STRING))) STORED, col2_12 STRING NOT NULL AS (CASE WHEN col2_̵1 IS NULL THEN '':::STRING ELSE e'T\x12#m3\x1aTtJ':::STRING END) STORED, INDEX (col2_7 DESC, col2_11) STORING (col2_3, col2_5, col2_6), UNIQUE (col2_̵1 DESC, (CASE WHEN col2_2 IS NULL THEN e'U{\x11{':::STRING ELSE 'XA':::STRING END) DESC, col2_8 DESC, col2_7 ASC, lower(CAST(col2_7 AS STRING)) DESC) STORING (col2_0, col2_2, col2_3, col2_5, "c ol2_9", col2_11), UNIQUE (col2_11, col2_̵1 DESC, col2_3, "c ol2_9" DESC, col2_8 ASC, col2_2 ASC) STORING (col2_5, col2_6, col2_7), INDEX (col2_̵1 ASC, col2_0 DESC) NOT VISIBLE, INDEX (col2_3 ASC, lower(CAST(col2_5 AS STRING)) DESC) STORING (col2_̵1, col2_5, "c ol2_9", col2_12) PARTITION BY LIST (col2_3) (PARTITION "t&😿able2_part0" VALUES IN (((-0.5844990611076355):::FLOAT8,), (0.6229314208030701:::FLOAT8,), ((-0.03161182254552841):::FLOAT8,), (0.4444349408149719:::FLOAT8,), (0.0316452756524086:::FLOAT8,), (0.5496028661727905:::FLOAT8,)), PARTITION "t&able😠2_part1" VALUES IN (((-0.15442249178886414):::FLOAT8,), ((-0.47291743755340576):::FLOAT8,), ((-1.0996148586273193):::FLOAT8,), (0.5906308889389038:::FLOAT8,), (0.600636899471283:::FLOAT8,), (0.4508877694606781:::FLOAT8,)), PARTITION "t&able2._part2" VALUES IN (((-0.8660415410995483):::FLOAT8,), (0.045772165060043335:::FLOAT8,), ((-0.2300938218832016):::FLOAT8,), ((-0.14880451560020447):::FLOAT8,), (1.401298464324817e-45:::FLOAT8,), (1.1305512189865112:::FLOAT8,)), PARTITION "t&able2_par%vt3" VALUES IN (((-0.4084572196006775):::FLOAT8,), ((-1.9808751344680786):::FLOAT8,), (2.261850595474243:::FLOAT8,), ((-0.8073147535324097):::FLOAT8,), ((-0.06638558954000473):::FLOAT8,), (2.1116244792938232:::FLOAT8,)), PARTITION "t&able2_part4" VALUES IN ((0.23162885010242462:::FLOAT8,), (0.057848889380693436:::FLOAT8,), ((-2.301356077194214):::FLOAT8,), (0.02630780078470707:::FLOAT8,), ((-1.6131610870361328):::FLOAT8,), ((-0.8181445002555847):::FLOAT8,)), PARTITION "t&able2_part5" VALUES IN ((0.9027278423309326:::FLOAT8,), (1.3771213293075562:::FLOAT8,), ((-1.8414281606674194):::FLOAT8,), ((-1.0):::FLOAT8,), (0.21204498410224915:::FLOAT8,), ((-2.856520891189575):::FLOAT8,)), PARTITION "t&able2_par t6" VALUES IN ((1.7974629402160645:::FLOAT8,), ((-1.0508254766464233):::FLOAT8,), ((-0.18642480671405792):::FLOAT8,), (0.14156238734722137:::FLOAT8,), ((-0.2472408562898636):::FLOAT8,), (0.9265791177749634:::FLOAT8,)), PARTITION "t&Able2_part7" VALUES IN (((-0.7237521409988403):::FLOAT8,), (1.1562812328338623:::FLOAT8,), ((-1.8223297595977783):::FLOAT8,), (0.6625334620475769:::FLOAT8,), ((-0.8352529406547546):::FLOAT8,), (1.6111681461334229:::FLOAT8,)), PARTITION "t&ab😯le2_parT8" VALUES IN (((-0.1949438750743866):::FLOAT8,), ((-0.10353291779756546):::FLOAT8,), ((-0.8776912689208984):::FLOAT8,), ((-0.07479815930128098):::FLOAT8,), ((-0.5571857690811157):::FLOAT8,), (0.5229619145393372:::FLOAT8,))), UNIQUE (col2_10 ASC, col2_11 ASC, "col2 _4" DESC, col2_2 ASC, col2_3 ASC, lower(CAST(col2_7 AS STRING)), (CASE WHEN col2_2 IS NULL THEN e'Y\x1b~\x12u\x157':::STRING ELSE e'X#\x1a\x06EN=%':::STRING END) DESC, col2_12 ASC, "c ol2_9") STORING (col2_5), UNIQUE (col2_5 ASC, col2_̵1, "col2 _4", col2_3 ASC, "c ol2_9" DESC, lower(CAST(col2_6 AS STRING)) DESC, col2_11, col2_6 ASC) WHERE (("t&able2".col2_11 >= '':::STRING) OR ("t&able2".col2_7 < '00:00:00':::TIME)) OR ("t&able2"."c ol2_9" <= 'X':::STRING));
    rsg_test.go:690: CREATE TABLE "table	3" (col3_0 VARCHAR NOT NULL, "col3\\u776E_1" BOOL NULL, "col3|_2" DATE NOT NULL, col3_3 TIME NOT NULL, col3_4 TSQUERY NOT NULL, "c o�l3_5" STRING NOT NULL AS (lower(col3_0)) STORED, UNIQUE ("col3\\u776E_1") WHERE ("table	3".col3_0 = 'X':::STRING) OR ("table	3"."col3|_2" = '-infinity':::DATE), UNIQUE (col3_3) WHERE ((("table	3"."col3\\u776E_1" OR ("table	3".col3_3 > '00:00:00':::TIME)) AND ("table	3"."col3|_2" = 'infinity':::DATE)) OR ("table	3"."c o�l3_5" > 'X':::STRING)) AND ("table	3".col3_0 != e'\U00002603':::STRING), UNIQUE ("c o�l3_5", "col3|_2" DESC) STORING (col3_3) WHERE (("table	3"."col3\\u776E_1" AND ("table	3".col3_3 >= '24:00:00':::TIME)) AND ("table	3".col3_0 = '':::STRING)) OR ("table	3"."col3|_2" > '4714-11-24 BC':::DATE));
    rsg_test.go:690: ALTER TABLE table1 INJECT STATISTICS e'[{"avg_size": 0, "columns": ["\\"c\\\\\\\\xf3ol1_0\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1119993, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "2007-06-26 03:33:30.000035+00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"avg_size": 8, "columns": ["\\"col%q1_1\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 356380, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"avg_size": 3, "columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4449763, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 600000, "num_eq": 5247162741637219890, "num_range": 600000, "upper_bound": "\\u0019~L"}, {"distinct_range": 100000, "num_eq": 60000, "num_range": 100000, "upper_bound": ".]``\\u00158N4Jv"}, {"distinct_range": 1846264979721327400, "num_eq": 8144014513829805690, "num_range": 5354590556719298744, "upper_bound": "1x|~"}, {"distinct_range": 6080391396684041000, "num_eq": 30000, "num_range": 6080391396684041225, "upper_bound": "Kc)dqQ"}, {"distinct_range": 9000000000, "num_eq": 9021806116016700801, "num_range": 9000000000, "upper_bound": "{y"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 6000000}]':::JSONB;
    rsg_test.go:690: ALTER TABLE "t&able2" INJECT STATISTICS e'[{"avg_size": 10, "columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 371203400, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"avg_size": 18, "columns": ["col2_\u03351"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 310646603, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "2004-02-01 07:31:22.000234"}, {"distinct_range": 3644717464924172300, "num_eq": 30, "num_range": 3644717464924172517, "upper_bound": "2018-01-23 02:52:13.000972"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"avg_size": 30, "columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 692899977, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "-1.512403964996338"}, {"distinct_range": 30000000, "num_eq": 80000, "num_range": 30000000, "upper_bound": "-0.7226042151451111"}, {"distinct_range": 43995985526.2099, "num_eq": 80000000000, "num_range": 50000000000, "upper_bound": "-0.6188451051712036"}, {"distinct_range": 2835290069080538000, "num_eq": 3935710774199798523, "num_range": 2835290069080538191, "upper_bound": "-0.3141712248325348"}, {"distinct_range": 3948631400188930000, "num_eq": 1080960849325228679, "num_range": 3948631400188929820, "upper_bound": "-0.00892520509660244"}, {"distinct_range": 703793444207906300, "num_eq": 0, "num_range": 1842039944678236116, "upper_bound": "0.28321734070777893"}, {"distinct_range": 492331098026876700, "num_eq": 1000, "num_range": 1334743974782838749, "upper_bound": "0.4818393886089325"}, {"distinct_range": 0, "num_eq": 2573326253954540686, "num_range": 800000, "upper_bound": "0.8130726218223572"}, {"distinct_range": 0, "num_eq": 4924184711027659779, "num_range": 800000000, "upper_bound": "1.905800461769104"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"avg_size": 3, "columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 504612721, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"avg_size": 20, "columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 667625622, "histo_buckets": [{"distinct_range": 0, "num_eq": 5793346147128278240, "num_range": 0, "upper_bound": "00:56:36.394753"}, {"distinct_range": 90, "num_eq": 200000000, "num_range": 90, "upper_bound": "01:13:48.655097"}, {"distinct_range": 3823771800620844000, "num_eq": 500, "num_range": 4004057076421141049, "upper_bound": "01:19:57.778888"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 90000000000, "upper_bound": "02:51:52.772447"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 8102248602438682908, "upper_bound": "06:40:50.375758"}, {"distinct_range": 0, "num_eq": 7623923361978703095, "num_range": 5570322453964197492, "upper_bound": "07:05:23.857026"}, {"distinct_range": 80000000000, "num_eq": 800, "num_range": 80000000000, "upper_bound": "09:24:11.233187"}, {"distinct_range": 0, "num_eq": 50, "num_range": 50000, "upper_bound": "15:45:40.735627"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"avg_size": 15, "columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 807704866, "histo_col_type": "", "name": "__auto__", "null_count": 503551151, "row_count": 1000000000}, {"avg_size": 9, "columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 582795101, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"avg_size": 17, "columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 351555610, "histo_buckets": [{"distinct_range": 0, "num_eq": 3028694884279995481, "num_range": 0, "upper_bound": "1\\u0003\\u0007"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"avg_size": 5, "columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 797067724, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"avg_size": 21, "columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 188432914, "histo_col_type": "", "name": "__auto__", "null_count": 335115323, "row_count": 1000000000}, {"avg_size": 10, "columns": ["\\"col2 _4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 864785373, "histo_col_type": "", "name": "__auto__", "null_count": 895965928, "row_count": 1000000000}, {"avg_size": 23, "columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 606123461, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"avg_size": 27, "columns": ["\\"c ol2_9\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 892431309, "histo_col_type": "", "name": "__auto__", "null_count": 355097979, "row_count": 1000000000}]':::JSONB;
    rsg_test.go:690: ALTER TABLE "table	3" INJECT STATISTICS e'[{"avg_size": 9, "columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19549462, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"avg_size": 29, "columns": ["\\"col3\\\\\\\\u776E_1\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 92184743, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 8561863532821540769, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 11947489, "row_count": 100000000}, {"avg_size": 26, "columns": ["\\"col3|_2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60903617, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"avg_size": 1, "columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53855058, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "00:03:40.507695"}, {"distinct_range": 2608758661632766000, "num_eq": 8743934205893081645, "num_range": 4871565779994278181, "upper_bound": "01:28:23.250779"}, {"distinct_range": 2072.678043501203, "num_eq": 0, "num_range": 4000, "upper_bound": "04:33:53.523484"}, {"distinct_range": 50444814.31060484, "num_eq": 100000000, "num_range": 60000000, "upper_bound": "05:52:45.54628"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 20, "upper_bound": "06:02:48.851653"}, {"distinct_range": 99.7514360355124, "num_eq": 10000000000, "num_range": 100, "upper_bound": "06:20:35.965865"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"avg_size": 2, "columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12925497, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"avg_size": 23, "columns": ["\\"c o\\u000bl3_5\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21156842, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}]':::JSONB;
    rsg_test.go:690: CREATE TYPE rand_typ_0 AS ENUM ('xj', 'jpvi', 'ctl', 'cfg', 'i', 'mgnme');
    rsg_test.go:690: CREATE TYPE rand_typ_1 AS ENUM ('gvh');
    rsg_test.go:690: CREATE TYPE rand_typ_2 AS ENUM ('c', 'rlle');
    rsg_test.go:690: CREATE TYPE rand_typ_3 AS ENUM ('rgzl', 'sbyfpp', 'lvo', 'fydw', 'of');
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: 
        BEGIN; CREATE TYPE greeting AS ENUM ('hello', 'howdy', 'hi', 'good day', 'morning'); COMMIT;
        BEGIN;
        CREATE TABLE IF NOT EXISTS seed AS
        	SELECT
        		g::INT2 AS _int2,
        		g::INT4 AS _int4,
        		g::INT8 AS _int8,
        		g::FLOAT4 AS _float4,
        		g::FLOAT8 AS _float8,
        		'2001-01-01'::DATE + g AS _date,
        		'2001-01-01'::TIMESTAMP + g * '1 day'::INTERVAL AS _timestamp,
        		'2001-01-01'::TIMESTAMPTZ + g * '1 day'::INTERVAL AS _timestamptz,
        		g * '1 day'::INTERVAL AS _interval,
        		g % 2 = 1 AS _bool,
        		g::DECIMAL AS _decimal,
        		g::STRING AS _string,
        		g::STRING::BYTES AS _bytes,
        		substring('00000000-0000-0000-0000-' || g::STRING || '00000000000', 1, 36)::UUID AS _uuid,
        		'0.0.0.0'::INET + g AS _inet,
        		g::STRING::JSONB AS _jsonb,
        		enum_range('hello'::greeting)[g] as _enum
        	FROM
        		generate_series(1, 5) AS g;
        COMMIT;
        
        INSERT INTO seed DEFAULT VALUES;
        CREATE INDEX on seed (_int8, _float8, _date);
        CREATE INVERTED INDEX on seed (_jsonb);
        ;
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (60518646352155718);
    rsg_test.go:690: ALTER TABLE seed SCATTER;
    rsg_test.go:692: 
    rsg_test.go:693: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxSQLSmith3440307605
--- FAIL: TestRandomSyntaxSQLSmith (304.04s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSQLSmith.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSQLSmith [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11410870?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11410870?buildTab=artifacts#/) on master @ [de0f60255e3f82da1b37504b294ae91f9268942d](https://github.com/cockroachdb/cockroach/commits/de0f60255e3f82da1b37504b294ae91f9268942d):

Random syntax error:

```
    rsg_test.go:880: Crash detected: server panic: statement exec timeout
```
Query:

```
        CREATE FUNCTION func429(IN p0 INET, IN p1 REGPROCEDURE, IN p2 INTERVAL, IN p3 REGTYPE, IN p4 REGPROC, IN p5 INT2VECTOR, IN p6 DECIMAL)
        	RETURNS SETOF RECORD
        	STRICT
        	LANGUAGE SQL
        	AS $funcbody$SELECT ",tab11667"._jsonb AS col29634, ",tab11667"._bool AS "Co̓l29635", ",tab11667"._decimal AS "Col29636", ",tab11667"._timestamp AS "%qcol͡29637", ARRAY[14249:::INT8,(-21334):::INT8,6139:::INT8,(-32768):::INT8] AS "c""😺ol29638" FROM defaultdb.public.seed@seed__int8__float8__date_idx AS ",tab11667" WHERE false ORDER BY ",tab11667"._string NULLS FIRST, ",tab11667"._decimal NULLS FIRST LIMIT 91:::INT8;
```
Schema:

```
    rsg_test.go:688: To reproduce, use schema:
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;;
    rsg_test.go:690: CREATE TABLE table1 (col1_0 TSVECTOR NOT NULL, col1_1 BYTES NOT NULL, "col1_\v2" GEOMETRY NOT NULL, col1_3 TSVECTOR NOT NULL, "c.ǒl1_4" CHAR NULL, INDEX (lower(CAST(col1_0 AS STRING)) ASC), UNIQUE ("c.ǒl1_4", col1_1) STORING (col1_0, "col1_\v2") WHERE table1.col1_1 > '\xff':::BYTES);
    rsg_test.go:690: CREATE TABLE table2 ("c\vOl̑2_""0" REGTYPE NULL, col2_1 TIMESTAMPTZ NOT NULL, "co'l2
        _2" TIMESTAMP, col2_3 REGCLASS NULL, col2_4 STRING COLLATE en_US NOT NULL, "co\\U000DFC20l2_5" STRING COLLATE en_US, "cOl2_6" CHAR NOT NULL, col2_7 INET NULL, col2_8 INET NOT NULL, col2_9 DECIMAL NOT NULL, col2_10 INT8 NULL, col2_11 PG_LSN NOT NULL, col2_12 STRING NOT NULL, col2_13 FLOAT4 NOT NULL, col2_14 STRING NOT NULL AS (lower("cOl2_6")) VIRTUAL, PRIMARY KEY (col2_4, col2_8 ASC), INDEX (col2_11 DESC, "cOl2_6" ASC, col2_7 ASC, col2_13 DESC, lower(CAST("co\\U000DFC20l2_5" AS STRING)), "c\vOl̑2_""0", col2_1 ASC, col2_3, col2_10 DESC, col2_14 ASC, col2_8 DESC, "co\\U000DFC20l2_5") WHERE (((((table2."cOl2_6" = e'\U00002603':::STRING) AND (table2.col2_14 != '':::STRING)) OR (table2."co'l2
        _2" = '2001-01-01 00:00:00 BC':::TIMESTAMP)) OR (table2.col2_12 > e'\x00':::STRING)) OR (table2.col2_9 <= (-1):::DECIMAL)) AND (table2.col2_1 <= '294276-12-31 23:59:59.999999+00':::TIMESTAMPTZ), UNIQUE (col2_1 ASC, lower("cOl2_6"), col2_4 DESC, col2_14 DESC, "co'l2
        _2" DESC, lower(CAST(col2_8 AS STRING)), col2_8 ASC, col2_11 ASC, col2_7 DESC, "co\\U000DFC20l2_5" DESC) PARTITION BY LIST (col2_1) (PARTITION table2_part😚0 VALUES IN (('1972-09-28 20:35:24.000734+00':::TIMESTAMPTZ,)), PARTITION "table%q2_part1" VALUES IN (('2029-09-12 13:43:34.000519+00':::TIMESTAMPTZ,)), PARTITION table2_part2 VALUES IN (('1995-07-21 08:03:34.000212+00':::TIMESTAMPTZ,)), PARTITION "DEFAULT" VALUES IN ((DEFAULT,))) WHERE (((((((table2.col2_14 >= '':::STRING) AND (table2.col2_9 >= '-Infinity':::DECIMAL)) OR (table2.col2_10 = 9223372036854775807:::INT8)) AND (table2.col2_1 <= '2001-01-01 00:00:00+00 BC':::TIMESTAMPTZ)) AND (table2.col2_12 = '':::STRING)) OR (table2.col2_13 < 'NaN':::FLOAT8)) OR (table2."cOl2_6" >= e'\'':::STRING)) AND (table2."co'l2
        _2" = '294276-12-31 23:59:59.999999':::TIMESTAMP), INDEX (col2_7 DESC, col2_1, col2_4, "c\vOl̑2_""0", col2_9 DESC, col2_11 ASC, "co'l2
        _2" ASC, "cOl2_6", col2_12 ASC, col2_14 ASC, col2_13 DESC, col2_8 ASC, "co\\U000DFC20l2_5", col2_3, col2_10), INDEX (col2_9) STORING (col2_1, "co'l2
        _2", "co\\U000DFC20l2_5", col2_7, col2_10, col2_11), UNIQUE (col2_14 DESC, col2_3 ASC, col2_1, "co\\U000DFC20l2_5") WHERE (((((((table2."co'l2
        _2" >= '2001-01-01 00:00:00 BC':::TIMESTAMP) OR (table2.col2_1 <= '4714-11-24 00:00:00+00 BC':::TIMESTAMPTZ)) OR (table2.col2_13 != '+Inf':::FLOAT8)) OR (table2.col2_12 >= '"':::STRING)) OR (table2.col2_14 > 'X':::STRING)) AND (table2.col2_9 = (-1):::DECIMAL)) AND (table2.col2_10 > 2147483647:::INT8)) OR (table2."cOl2_6" >= '':::STRING), INDEX ("c\vOl̑2_""0", col2_13 ASC, col2_9 ASC, col2_14 DESC, col2_7 DESC, col2_10 ASC) STORING (col2_1, "co\\U000DFC20l2_5", "cOl2_6", col2_12), UNIQUE ((CASE WHEN col2_3 IS NULL THEN ';?9ZP6``w':::STRING ELSE e'\x18\x02X':::STRING END) DESC, col2_14) STORING (col2_1, "co'l2
        _2", col2_3, "cOl2_6", col2_7, col2_10, col2_11, col2_12), UNIQUE ("c\vOl̑2_""0", col2_11, "co\\U000DFC20l2_5" DESC, col2_13 ASC, col2_1 DESC, (col2_9 + 8.198244255401527385E+28:::DECIMAL) DESC) STORING ("cOl2_6", col2_10) WHERE (((table2.col2_13 < 0.0:::FLOAT8) OR (table2."co'l2
        _2" != '0001-01-01 00:00:00':::TIMESTAMP)) AND (table2."cOl2_6" < 'X':::STRING)) AND (table2.col2_14 >= '"':::STRING), UNIQUE (col2_9, col2_12 DESC, col2_7, col2_10 ASC, "co'l2
        _2" ASC, "c\vOl̑2_""0", col2_14 ASC, col2_3, col2_13 DESC, lower("cOl2_6") DESC, col2_11) WHERE ((((((table2.col2_1 < '4714-11-24 00:00:00+00 BC':::TIMESTAMPTZ) AND (table2.col2_13 <= 0.0:::FLOAT8)) OR (table2."co'l2
        _2" = '3000-01-01 00:00:00':::TIMESTAMP)) AND (table2.col2_9 >= 'Infinity':::DECIMAL)) OR (table2.col2_10 != 9223372036854775807:::INT8)) OR (table2.col2_14 >= 'X':::STRING)) OR (table2."cOl2_6" = '':::STRING));
    rsg_test.go:690: ALTER TABLE table1 INJECT STATISTICS e'[{"avg_size": 29, "columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6531081595864519894, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8222986919892825914}, {"avg_size": 16, "columns": ["\\"c.\u01D2l1_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4926315786161121493, "histo_buckets": [{"distinct_range": 0, "num_eq": 5805703559314271278, "num_range": 0, "upper_bound": "\\u0010"}, {"distinct_range": 0, "num_eq": 431374216488672840, "num_range": 100000000, "upper_bound": "\\u0019"}, {"distinct_range": 313446114.40235686, "num_eq": 2439187310666168497, "num_range": 700000000, "upper_bound": "\\""}, {"distinct_range": 236504950.248766, "num_eq": 7292834634780962502, "num_range": 300000000, "upper_bound": "2"}, {"distinct_range": 295860322920505000, "num_eq": 700000000, "num_range": 6010848366470780847, "upper_bound": "6"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 3438133404798489047, "row_count": 8222986919892825914}, {"avg_size": 24, "columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3713376839355751115, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8222986919892825914}, {"avg_size": 5, "columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6878202707503472615, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8222986919892825914}, {"avg_size": 16, "columns": ["\\"col1_\\\\v2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7932887248214734045, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8222986919892825914}]':::JSONB;
    rsg_test.go:690: ALTER TABLE table2 INJECT STATISTICS e'[{"avg_size": 15, "columns": ["\\"cOl2_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4236859000419901168, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9031957714117483509}, {"avg_size": 18, "columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4325706326992211161, "histo_buckets": [{"distinct_range": 0, "num_eq": 1236778688288416465, "num_range": 0, "upper_bound": "13.33.0.142/11"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 6000000000, "upper_bound": "56a:d34a:5c6e:c2a4:7bae:de9:889a:594a/83"}, {"distinct_range": 30000000000, "num_eq": 2772851671440364701, "num_range": 30000000000, "upper_bound": "f8bb:92ee:9d11:4ee6:135d:b238:820e:7bf3/120"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 5068399121755285049, "row_count": 9031957714117483509}, {"avg_size": 8, "columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4559798595639065964, "histo_col_type": "PG_LSN", "name": "__auto__", "null_count": 0, "row_count": 9031957714117483509}, {"avg_size": 31, "columns": ["\\"co\\\\\\\\U000DFC20l2_5\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7313678381336319229, "histo_col_type": "", "name": "__auto__", "null_count": 1545981482694870176, "row_count": 9031957714117483509}, {"avg_size": 1, "columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8137857424147479270, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9031957714117483509}, {"avg_size": 11, "columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6175514798333421988, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9031957714117483509}, {"avg_size": 2, "columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5702563630998278981, "histo_buckets": [{"distinct_range": 0, "num_eq": 6165835765379370424, "num_range": 0, "upper_bound": ""}, {"distinct_range": 5000000, "num_eq": 90000000000, "num_range": 5000000, "upper_bound": "_\\"\\n\\u0019"}, {"distinct_range": 5460535990552099000, "num_eq": 1119441752435639239, "num_range": 5460535990552098450, "upper_bound": "\U00002603"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 9031957714117483509}, {"avg_size": 25, "columns": ["\\"c\\\\vOl\u03112_\\"\\"0\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1306879554198296829, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "3011158386"}, {"distinct_range": 0, "num_eq": 6686513168537527102, "num_range": 924297091816029863, "upper_bound": "3332234544"}, {"distinct_range": 778397998949839100, "num_eq": 5840625249616649146, "num_range": 2406123520350274354, "upper_bound": "3566699730"}, {"distinct_range": 0, "num_eq": 90, "num_range": 606932713586994198, "upper_bound": "3659275223"}, {"distinct_range": 4049607556244033000, "num_eq": 9000, "num_range": 5103715161690993283, "upper_bound": "3684045814"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 10000000, "upper_bound": "4111010033"}, {"distinct_range": 35.41631140200939, "num_eq": 0, "num_range": 70, "upper_bound": "4261925895"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 3577716697306090569, "row_count": 9031957714117483509}, {"avg_size": 17, "columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6083086236589228416, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "3000-01-01 00:00:00+00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 9031957714117483509}, {"avg_size": 26, "columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3152178932178239871, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9031957714117483509}, {"avg_size": 23, "columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1936370725828071117, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "2.465739213786182287E+26"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 9031957714117483509}, {"avg_size": 19, "columns": ["\\"co\'l2\\n_2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4167165267796526489, "histo_col_type": "", "name": "__auto__", "null_count": 1265140869233540468, "row_count": 9031957714117483509}, {"avg_size": 23, "columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7099380801667684504, "histo_col_type": "", "name": "__auto__", "null_count": 7025116339322856085, "row_count": 9031957714117483509}, {"avg_size": 27, "columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2854893866824712612, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9031957714117483509}, {"avg_size": 22, "columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1466056104176984768, "histo_col_type": "", "name": "__auto__", "null_count": 590337035034270059, "row_count": 9031957714117483509}]':::JSONB;
    rsg_test.go:690: CREATE TYPE rand_typ_0 AS ENUM ('uxz', 'auatnp', 'x');
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: 
        BEGIN; CREATE TYPE greeting AS ENUM ('hello', 'howdy', 'hi', 'good day', 'morning'); COMMIT;
        BEGIN;
        CREATE TABLE IF NOT EXISTS seed AS
        	SELECT
        		g::INT2 AS _int2,
        		g::INT4 AS _int4,
        		g::INT8 AS _int8,
        		g::FLOAT4 AS _float4,
        		g::FLOAT8 AS _float8,
        		'2001-01-01'::DATE + g AS _date,
        		'2001-01-01'::TIMESTAMP + g * '1 day'::INTERVAL AS _timestamp,
        		'2001-01-01'::TIMESTAMPTZ + g * '1 day'::INTERVAL AS _timestamptz,
        		g * '1 day'::INTERVAL AS _interval,
        		g % 2 = 1 AS _bool,
        		g::DECIMAL AS _decimal,
        		g::STRING AS _string,
        		g::STRING::BYTES AS _bytes,
        		substring('00000000-0000-0000-0000-' || g::STRING || '00000000000', 1, 36)::UUID AS _uuid,
        		'0.0.0.0'::INET + g AS _inet,
        		g::STRING::JSONB AS _jsonb,
        		enum_range('hello'::greeting)[g] as _enum
        	FROM
        		generate_series(1, 5) AS g;
        COMMIT;
        
        INSERT INTO seed DEFAULT VALUES;
        CREATE INDEX on seed (_int8, _float8, _date);
        CREATE INVERTED INDEX on seed (_jsonb);
        ;
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (7558381831002766372);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (3425553020096465710);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (9010547755159726799);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (2099829026022875914);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (3074994300888998568);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (3608297978871915808);
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (3434257014710304207);
    rsg_test.go:690: ALTER TABLE seed SCATTER;
    rsg_test.go:692: 
    rsg_test.go:693: -- test log scope end --
--- FAIL: TestRandomSyntaxSQLSmith (303.12s)
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxSQLSmith337075110
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSQLSmith.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSQLSmith [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11429792?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11429792?buildTab=artifacts#/) on master @ [dc56824a3f7b1d90b398073fab82120cded0bd9c](https://github.com/cockroachdb/cockroach/commits/dc56824a3f7b1d90b398073fab82120cded0bd9c):

Random syntax error:

```
    rsg_test.go:880: Crash detected: server panic: statement exec timeout
```
Query:

```
        CREATE FUNCTION "%1cfun{c232"(IN p0 BIT(34), IN p1 PG_LSN, IN p2 INT2)
        	RETURNS REGROLE
        	VOLATILE
        	NOT LEAKPROOF
        	LANGUAGE SQL
        	AS $funcbody$SELECT (-13538):::INT8 AS "cOL15422", (3440426643083108534:::INT8::INT8 % NULL::INT8)::INT8 AS col15423, NULL AS "c ol15424", tab6416._enum AS "co)
l15425", uuid(tab6416._uuid::UUID)::UUID AS col15426, tab6416._int2 AS col15427, tab6416._decimal AS col15428, timezone(tab6416._string::STRING, NULL::STRING)::TIMESTAMP AS col15429, tab6416._bool AS col15430, tab6416._inet AS "coL15431", p0 AS col😷15432 FROM defaultdb.public.seed@[0] AS tab6416 LIMIT 95:::INT8;
```
Schema:

```
    rsg_test.go:688: To reproduce, use schema:
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;;
    rsg_test.go:690: SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;;
    rsg_test.go:690: CREATE TABLE table1 (col1_0 TIMESTAMP NOT NULL, col1_1 TIMESTAMP NOT NULL, col1̟_2 OID NOT NULL, col1_3 REGROLE NOT NULL, col1_4 BIT(34) NOT NULL, " col1_ 5" DATE NOT NULL, "coL1_6" PG_LSN NOT NULL, "col1%p_7" BIT(32) NOT NULL, "{c ol1_8" INET NOT NULL, PRIMARY KEY ("col1%p_7" DESC, col1_4 DESC, col1̟_2 ASC, " col1_ 5", "{c ol1_8" ASC, col1_0 DESC, col1_3, col1_1 DESC, "coL1_6"), UNIQUE (col1_3, col1̟_2 DESC, col1_0 ASC, lower(CAST(col1_4 AS STRING)) DESC) WHERE ((table1.col1_0 <= '3000-01-01 00:00:00':::TIMESTAMP) AND (table1." col1_ 5" = '-infinity':::DATE)) OR (table1.col1_1 > '3000-01-01 00:00:00':::TIMESTAMP), INDEX ("coL1_6" DESC, "col1%p_7" ASC, col1_3 DESC) WHERE ((table1.col1_1 >= '3000-01-01 00:00:00':::TIMESTAMP) AND (table1.col1_0 != '2001-01-01 00:00:00 BC':::TIMESTAMP)) OR (table1." col1_ 5" >= '1970-01-01':::DATE), FAMILY (col1_4, col1_3, "{c ol1_8"), FAMILY (col1_1), FAMILY (col1̟_2, "col1%p_7"), FAMILY (" col1_ 5", col1_0), FAMILY ("coL1_6"));
    rsg_test.go:690: CREATE TABLE table2 (col2_0 BIT(7) NOT NULL, col2_1 INT2 NOT NULL, "c ol2_̵2" DECIMAL NOT NULL, UNIQUE ("c ol2_̵2" DESC) STORING (col2_1) PARTITION BY LIST ("c ol2_̵2") (PARTITION "Table2_part0" VALUES IN ((8.958424994409151525E+34:::DECIMAL,), ((-6.933035045836130591E+22):::DECIMAL,), (569.6856821308885418:::DECIMAL,)), PARTITION table2_part1 VALUES IN (((-7.015804653607308664E+26):::DECIMAL,), (4.341985690043678756E+26:::DECIMAL,), ((-66.7388516604355893):::DECIMAL,)), PARTITION table2_part2 VALUES IN ((8925823928.424064061:::DECIMAL,), ((-319.1604027995931649):::DECIMAL,), ((-4.048053884174179496E+26):::DECIMAL,)), PARTITION "table\\x652_part3" VALUES IN ((0:::DECIMAL,), (7.797673772264754967E+20:::DECIMAL,), ((-7.470196542495981311E+35):::DECIMAL,)), PARTITION "tabl�e2_p|art4" VALUES IN (((-70111714250453773.23):::DECIMAL,), ((-3.492556821371386660E+24):::DECIMAL,), (4135502574373.826350:::DECIMAL,))), INDEX ("c ol2_̵2" DESC, col2_0 DESC, col2_1 ASC) NOT VISIBLE, UNIQUE (col2_0, "c ol2_̵2", col2_1), UNIQUE (col2_0 DESC, ("c ol2_̵2" + 890547552057697564.8:::DECIMAL) ASC, "c ol2_̵2" ASC), UNIQUE (col2_0 ASC, col2_1) STORING ("c ol2_̵2"), INDEX (col2_1 ASC, "c ol2_̵2", col2_0 ASC) WHERE (table2."c ol2_̵2" <= (-1):::DECIMAL) AND (table2.col2_1 <= 1:::INT8) NOT VISIBLE);
    rsg_test.go:690: CREATE TABLE table3 (col3_0 INT4 NOT NULL, col3_1 STRING NOT NULL, "co|😎l3_2" DATE[] NOT NULL, col3͜_3 FLOAT4 NOT NULL, "col3�_4" UUID NOT NULL, col3_5 STRING NOT NULL, "c 😋ol3_6" CHAR NULL, col3_7 INT2 NOT NULL, col3_8 FLOAT8 NOT NULL, "col3""_9" INT8 NOT NULL AS (col3_7 + col3_0) STORED, col3_10 FLOAT8 NOT NULL AS (col3͜_3 + col3_8) VIRTUAL, cô😕l3_11 INT8 NOT NULL AS (col3_7 + (-1):::INT8) STORED, co͈l😐3_12 FLOAT8 NOT NULL AS (col3͜_3 + col3_8) VIRTUAL, col3_13 FLOAT8 NOT NULL AS (col3_8 + col3͜_3) STORED, col3_14 FLOAT8 NOT NULL AS (col3_8 + col3͜_3) STORED, col3😿_15 STRING NOT NULL AS (CASE WHEN "co|😎l3_2" IS NULL THEN '@':::STRING ELSE e'\x01!{l\fF!p':::STRING END) STORED, "co|l3_16" STRING NOT NULL AS (lower(col3_1)) VIRTUAL, PRIMARY KEY (col3_1, col3_10 DESC, col3_7 DESC, "col3""_9", "co|😎l3_2" ASC, co͈l😐3_12 ASC), UNIQUE (lower("c 😋ol3_6") DESC, col3_7), INDEX (col3_8 ASC, (col3_0 + col3_7) ASC, "col3""_9" ASC, col3_5 ASC, "co|l3_16", (col3͜_3 + col3_8) ASC, col3_13 DESC, col3͜_3, "co|😎l3_2" ASC, "col3�_4", cô😕l3_11 DESC, col3_7 ASC, "c 😋ol3_6", col3_0 DESC, col3_10 DESC) WHERE (table3."col3""_9" > 32767:::INT8) AND (table3."c 😋ol3_6" >= '':::STRING) VISIBILITY 0.38, UNIQUE (col3͜_3, "c 😋ol3_6", (col3_0 + (-32768):::INT8), col3_1 DESC), INDEX (cô😕l3_11 DESC) STORING ("col3�_4", col3_5, col3_8, col3_13) PARTITION BY LIST (cô😕l3_11) (PARTITION "table3_p(Art0" VALUES IN ((9110570681893690218:::INT8,), (9194237311047389406:::INT8,), ((-2147483648):::INT8,), (8516118062760012705:::INT8,), (1010582709913969496:::INT8,), ((-6200898948981102990):::INT8,)), PARTITION table3_part1 VALUES IN (((-2033008202923828302):::INT8,), ((-6427660305321715714):::INT8,), (7519953104939637019:::INT8,), (6584592612764288841:::INT8,), (3992611763392841365:::INT8,), ((-8975102902673135238):::INT8,)), PARTITION table3😐_part2 VALUES IN ((5276784289369518175:::INT8,), (8846117554966238163:::INT8,), (5734011284514229745:::INT8,), ((-625448899570114648):::INT8,), (4770389214653136134:::INT8,), (8742350091243028188:::INT8,)), PARTITION table3_part3 VALUES IN (((-2101756165337171455):::INT8,), (2416635962015475363:::INT8,), (2405817724111318320:::INT8,), ((-6098895534341530112):::INT8,), (5699954505400617792:::INT8,), (2905393769193865758:::INT8,)), PARTITION table3_part4 VALUES IN ((495663815772003455:::INT8,), ((-4829435732555414259):::INT8,), ((-5385723808491043813):::INT8,), (5357327732188764483:::INT8,), (1098318440259693399:::INT8,), (1093675131782412609:::INT8,)), PARTITION table3_p😳art5 VALUES IN (((-1):::INT8,), (7809991568545936954:::INT8,), (7047555809309037565:::INT8,), (6968825602285274207:::INT8,), ((-3169829942651492018):::INT8,)), PARTITION "ta'ble3_part/6" VALUES IN (((-323948493859938077):::INT8,), ((-5410207854326996378):::INT8,), (6926249900443045856:::INT8,), ((-3480444265020333346):::INT8,), ((-87466367575112784):::INT8,), (8689283179639217110:::INT8,)), PARTITION table3_part7 VALUES IN (((-7640083235372613027):::INT8,), ((-468648251608471580):::INT8,), (5212492372415374634:::INT8,), ((-7196554834370700202):::INT8,), ((-5250164351276488119):::INT8,), ((-5405386665538977718):::INT8,)), PARTITION "DEFAULT" VALUES IN ((DEFAULT,))), UNIQUE (col3_5 DESC, "col3""_9" ASC, (col3_8 + (-0.8498778119337):::FLOAT8) ASC, col3͜_3 DESC, col3_14, lower("c 😋ol3_6") ASC, col3😿_15, "co|l3_16" ASC, "col3�_4" ASC) WHERE (((((((((((((table3.col3_1 > 'X':::STRING) OR (table3."c 😋ol3_6" != e'\'':::STRING)) AND (table3.col3_7 > 1:::INT8)) OR (table3.cô😕l3_11 >= (-9223372036854775807):::INT8)) AND (table3.col3😿_15 >= '':::STRING)) OR (table3.col3_14 = 3.4028234663852886e+38:::FLOAT8)) AND (table3.col3͜_3 > '+Inf':::FLOAT8)) AND (table3.col3_13 <= 'NaN':::FLOAT8)) OR (table3."col3""_9" > 9223372036854775807:::INT8)) OR (table3.col3_10 >= '-Inf':::FLOAT8)) OR (table3.col3_8 != 1.7976931348623157e+308:::FLOAT8)) OR (table3."co|l3_16" <= '':::STRING)) OR (table3.co͈l😐3_12 != (-1.0):::FLOAT8)) OR (table3.col3_5 != e'\U00002603':::STRING), INDEX (col3_7 ASC, "col3�_4", "c 😋ol3_6" DESC, col3_8, col3😿_15 ASC, "col3""_9", col3_13, col3_5 ASC, col3͜_3 DESC, (col3_8 + 0.7399370561926809:::FLOAT8) DESC, cô😕l3_11 DESC, co͈l😐3_12 ASC, col3_14 DESC, (col3_0 + col3_7) ASC, "co|l3_16", col3_1 ASC, "co|😎l3_2") WHERE (((table3.col3_7 <= 1:::INT8) AND (table3."co|l3_16" > 'X':::STRING)) AND (table3.cô😕l3_11 < 1:::INT8)) OR (table3.col3_0 > (-1):::INT8), INDEX ("col3""_9", (col3_8 + col3͜_3), col3_1 ASC, col3_8 ASC, col3_13 DESC, col3_0 ASC, col3_10, co͈l😐3_12 ASC, (col3_0 + col3_7)) STORING ("c 😋ol3_6", col3_14), UNIQUE (col3͜_3, (CASE WHEN "co|😎l3_2" IS NULL THEN '"':::STRING ELSE e'9[4``\x05D>\x05\x18':::STRING END) ASC, "co|😎l3_2" DESC, col3_8, (col3_7 + col3_0) DESC, col3_1, col3_7 DESC, (col3_8 + col3͜_3) ASC, col3_13, lower(CAST("col3�_4" AS STRING)) ASC, col3_5, "col3""_9", col3_10 ASC) WHERE table3.co͈l😐3_12 <= '+Inf':::FLOAT8, INDEX ((col3_7 + col3_0) ASC, "col3""_9" ASC, col3_10 DESC, (col3_8 + col3͜_3) DESC), FAMILY (col3͜_3), FAMILY (col3_8), FAMILY (col3_7), FAMILY ("c 😋ol3_6"), FAMILY ("col3""_9", col3_1), FAMILY (col3😿_15), FAMILY (cô😕l3_11), FAMILY (col3_0), FAMILY (col3_14), FAMILY ("co|😎l3_2", col3_13, "col3�_4"), FAMILY (col3_5));
    rsg_test.go:690: ALTER TABLE table1 INJECT STATISTICS e'[{"avg_size": 28, "columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 845953707549419922, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7697594232405328835}, {"avg_size": 0, "columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1344553922263678029, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7697594232405328835}, {"avg_size": 17, "columns": ["\\"col1%p_7\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1657169790060292479, "histo_buckets": [{"distinct_range": 0, "num_eq": 1425088968051072297, "num_range": 0, "upper_bound": "00000000000000000000000000000000"}, {"distinct_range": 0, "num_eq": 4103378138699057340, "num_range": 2000, "upper_bound": "00100101101010010000001000000100"}, {"distinct_range": 0, "num_eq": 3894346473968531923, "num_range": 8363705824587044747, "upper_bound": "10100101011011011110001010011101"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 1125116561340173530, "upper_bound": "11010101010000101100110111100111"}, {"distinct_range": 10000000, "num_eq": 8410146587391868455, "num_range": 10000000, "upper_bound": "11110110011100100010001101110101"}], "histo_col_type": "BIT(32)", "name": "__auto__", "null_count": 0, "row_count": 7697594232405328835}, {"avg_size": 24, "columns": ["\\"{c ol1_8\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3379020332029713564, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7697594232405328835}, {"avg_size": 29, "columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3856105090923117750, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7697594232405328835}, {"avg_size": 2, "columns": ["col1\u031F_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1641094123974515949, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7697594232405328835}, {"avg_size": 11, "columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6907658797063159561, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1473342954"}, {"distinct_range": 1326619139497975000, "num_eq": 5304228057561765053, "num_range": 2557216075844292409, "upper_bound": "1947694685"}, {"distinct_range": 500000000, "num_eq": 600000, "num_range": 500000000, "upper_bound": "2595031105"}, {"distinct_range": 5320348.5183759015, "num_eq": 900, "num_range": 7000000, "upper_bound": "2867422151"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "3164614753"}, {"distinct_range": 0, "num_eq": 8353601573789843700, "num_range": 2304292071618925201, "upper_bound": "3225348754"}, {"distinct_range": 2860734443606802400, "num_eq": 2636783344551086567, "num_range": 2860734443606802342, "upper_bound": "3645151837"}, {"distinct_range": 0, "num_eq": 6758692965842322477, "num_range": 744246848810875183, "upper_bound": "4281275366"}], "histo_col_type": "REGROLE", "name": "__auto__", "null_count": 0, "row_count": 7697594232405328835}, {"avg_size": 8, "columns": ["\\" col1_ 5\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4739683969859322065, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7697594232405328835}, {"avg_size": 12, "columns": ["\\"coL1_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6651941926601484560, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7697594232405328835}]':::JSONB;
    rsg_test.go:690: ALTER TABLE table2 INJECT STATISTICS e'[{"avg_size": 19, "columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1293645371211631814, "histo_col_type": "BIT(7)", "name": "__auto__", "null_count": 0, "row_count": 2329209650688783258}, {"avg_size": 29, "columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1540556926717289354, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "10726"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 2329209650688783258}, {"avg_size": 1, "columns": ["\\"c ol2_\u03352\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1837135298432829739, "histo_buckets": [{"distinct_range": 0, "num_eq": 7477789735962984096, "num_range": 0, "upper_bound": "-7.241252638324307758E+30"}, {"distinct_range": 100000, "num_eq": 5369785755183983525, "num_range": 100000, "upper_bound": "-9.36119835345645459E+26"}, {"distinct_range": 9464049291.661894, "num_eq": 7084430919672406573, "num_range": 10000000000, "upper_bound": "-956187625794841385"}, {"distinct_range": 858801699779817500, "num_eq": 200, "num_range": 3528109446446325207, "upper_bound": "-78531575.23779926606"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 6410540848837159523, "upper_bound": "-4908.347535787106832"}, {"distinct_range": 5204569687260962000, "num_eq": 5020724825190551706, "num_range": 5204569687260961660, "upper_bound": "1.613680394165913349"}, {"distinct_range": 5547953363.8424635, "num_eq": 1138189617588718011, "num_range": 9000000000, "upper_bound": "2487223680176380.566"}, {"distinct_range": 650077721486972500, "num_eq": 90000000, "num_range": 5044860204097882338, "upper_bound": "5.298313251871604852E+24"}, {"distinct_range": 4513032538843695600, "num_eq": 2432120365234602392, "num_range": 7377568621068392430, "upper_bound": "8.631468405009778276E+30"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 2329209650688783258}]':::JSONB;
    rsg_test.go:690: ALTER TABLE table3 INJECT STATISTICS e'[{"avg_size": 8, "columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11251, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 3, "columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31850, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 28, "columns": ["col3\U0001F63F_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22012, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 1, "columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48815, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 6, "columns": ["\\"c \U0001F60Bol3_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9949, "histo_col_type": "", "name": "__auto__", "null_count": 48018, "row_count": 60000}, {"avg_size": 22, "columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29610, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 23, "columns": ["\\"col3\\"\\"_9\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40604, "histo_buckets": [{"distinct_range": 0, "num_eq": 1254189991753574789, "num_range": 0, "upper_bound": "-7839143819391484472"}, {"distinct_range": 53658.95191771579, "num_eq": 3052950645882297602, "num_range": 100000, "upper_bound": "-2768658692349676119"}, {"distinct_range": 400000000, "num_eq": 4272014799301867539, "num_range": 400000000, "upper_bound": "3478045548398875239"}, {"distinct_range": 8521177199453218000, "num_eq": 100000000000, "num_range": 8521177199453217805, "upper_bound": "4753736418898540371"}, {"distinct_range": 0, "num_eq": 8496469069035269348, "num_range": 6177525674499460470, "upper_bound": "5905423315168247692"}, {"distinct_range": 1417389042534826500, "num_eq": 7973746802281903125, "num_range": 8826517935870127418, "upper_bound": "9119047971929005400"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 15, "columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50650, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 18, "columns": ["\\"co|\U0001F60El3_2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30392, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 9, "columns": ["\\"col3\\u000b_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56395, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 10, "columns": ["c\u00F4\U0001F615l3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6854, "histo_buckets": [{"distinct_range": 0, "num_eq": 6920354977675353118, "num_range": 0, "upper_bound": "-8851925366839996889"}, {"distinct_range": 57.702414200150486, "num_eq": 7597600140609408422, "num_range": 60, "upper_bound": "-4003884427010261311"}, {"distinct_range": 78895.00590009594, "num_eq": 6059388057903205039, "num_range": 900000, "upper_bound": "-1134860448113771128"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "-65919605518938684"}, {"distinct_range": 1000000000, "num_eq": 696067801501653020, "num_range": 1000000000, "upper_bound": "32767"}, {"distinct_range": 37895.21465547723, "num_eq": 9000, "num_range": 80000, "upper_bound": "886266768423111458"}, {"distinct_range": 0, "num_eq": 2074010272464113402, "num_range": 5869203631932756442, "upper_bound": "1133453428888442335"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 7823395673212860738, "upper_bound": "2625255175524608049"}, {"distinct_range": 0, "num_eq": 4670454093527146013, "num_range": 1250955277538416179, "upper_bound": "3970229425671505069"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 2, "columns": ["co\u0348l\U0001F6103_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 54967, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 0, "columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40175, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 5, "columns": ["\\"co|l3_16\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30306, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 1, "columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39874, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 21, "columns": ["col3\u035C_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33064, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"avg_size": 19, "columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52500, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}]':::JSONB;
    rsg_test.go:690: CREATE TYPE rand_typ_0 AS ENUM ('bzybx', 'riwqa', 'uhj', 'x', 'r');
    rsg_test.go:690: CREATE TYPE rand_typ_1 AS ENUM ('ru', 'nneyi', 'xtv', 'tqzo');
    rsg_test.go:690: CREATE TYPE rand_typ_2 AS ENUM ('v', 'lmcphx', 'sp', 'eetum');
    rsg_test.go:690: CREATE TYPE rand_typ_3 AS ENUM ('myxvoq', 'hnp', 'x', 'nozsm', 'vezrfw', 'ysfdje');
    rsg_test.go:690: CREATE TYPE rand_typ_4 AS ENUM ('xz', 'gio', 'sx', 'drdkgm');
    rsg_test.go:690: SET sql_safe_updates = false;;
    rsg_test.go:690: 
        BEGIN; CREATE TYPE greeting AS ENUM ('hello', 'howdy', 'hi', 'good day', 'morning'); COMMIT;
        BEGIN;
        CREATE TABLE IF NOT EXISTS seed AS
        	SELECT
        		g::INT2 AS _int2,
        		g::INT4 AS _int4,
        		g::INT8 AS _int8,
        		g::FLOAT4 AS _float4,
        		g::FLOAT8 AS _float8,
        		'2001-01-01'::DATE + g AS _date,
        		'2001-01-01'::TIMESTAMP + g * '1 day'::INTERVAL AS _timestamp,
        		'2001-01-01'::TIMESTAMPTZ + g * '1 day'::INTERVAL AS _timestamptz,
        		g * '1 day'::INTERVAL AS _interval,
        		g % 2 = 1 AS _bool,
        		g::DECIMAL AS _decimal,
        		g::STRING AS _string,
        		g::STRING::BYTES AS _bytes,
        		substring('00000000-0000-0000-0000-' || g::STRING || '00000000000', 1, 36)::UUID AS _uuid,
        		'0.0.0.0'::INET + g AS _inet,
        		g::STRING::JSONB AS _jsonb,
        		enum_range('hello'::greeting)[g] as _enum
        	FROM
        		generate_series(1, 5) AS g;
        COMMIT;
        
        INSERT INTO seed DEFAULT VALUES;
        CREATE INDEX on seed (_int8, _float8, _date);
        CREATE INVERTED INDEX on seed (_jsonb);
        ;
    rsg_test.go:690: ALTER TABLE seed SPLIT AT VALUES (6733268316567972926);
    rsg_test.go:690: ALTER TABLE seed SCATTER;
    rsg_test.go:692: 
    rsg_test.go:693: -- test log scope end --
--- FAIL: TestRandomSyntaxSQLSmith (302.11s)
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxSQLSmith22057989
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSQLSmith.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxSQLSmith [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11449053?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/11449053?buildTab=artifacts#/) on master @ [0bbf845a525d02661b853a4dd1190d8b9831a3d4](https://github.com/cockroachdb/cockroach/commits/0bbf845a525d02661b853a4dd1190d8b9831a3d4):

Random syntax error:

```
    rsg_test.go:887: Crash detected: server panic: statement exec timeout
```
Query:

```
        CREATE FUNCTION "fun\\U000B1C88c28"(IN p0 greeting, IN p1 BOX2D[], IN p2 VARCHAR, IN p3 BIT(27), IN p4 TIME, IN p5 UUID)
        	RETURNS REGPROC
        	NOT LEAKPROOF
        	LANGUAGE SQL
        	AS $funcbody$SELECT tab742.crdb_internal_mvcc_timestamp AS "co'😈l1801", tab742._timestamptz AS col1802 FROM defaultdb.public.seed@[0] AS tab740, defaultdb.public.seed@[0] AS "\rtab741" JOIN defaultdb.public.seed@[0] AS tab742 ON ((((((("\rtab741"._int2) = (tab742._int4)) AND (("\rtab741"._decimal) = (tab742._decimal))) AND (("\rtab741"._int8) = (tab742._int2))) AND (("\rtab741"._int8) = (tab742._int8))) AND (("\rtab741"._float8) = (tab742._float8))) AND (("\rtab741"._inet) = (tab742._inet))) AND (("\rtab741"._int4) = (tab742._int2)) WHERE ('0.0.0.0/0':::INET::INET << '1a95:20b:da5c:fa7a:936:38cb:6b25:7e1b/25':::INET::INET)::BOOL ORDER BY tab742._date NULLS LAST, tab742._bool ASC LIMIT 72:::INT8;
```
Schema:

```
    rsg_test.go:695: To reproduce, use schema:
    rsg_test.go:697: SET sql_safe_updates = false;;
    rsg_test.go:697: SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;;
    rsg_test.go:697: SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;;
    rsg_test.go:697: CREATE TABLE table1 (c͈ol1_0 REGCLASS, "col1̈_ 1" STRING NOT NULL, col1_2 INT2 NOT NULL, "c'ol1_%p3" PG_LSN NOT NULL, "Col1_4" BIT(46) NOT NULL, "col1 _5" REGCLASS NOT NULL, "cOl1_6" BOX2D NOT NULL, "c%vo\\x1bl1_\\x117" BOX2D NOT NULL, col1😪_8 TIMESTAMPTZ NOT NULL, col1_9 REGROLE NOT NULL, "c😙ol̟1_%10" INTERVAL NOT NULL, col1_11 STRING NOT NULL AS (lower(CAST("cOl1_6" AS STRING))) STORED, col1_12 INT8 NOT NULL AS (col1_2 + 18633:::INT8) STORED, col1_13 STRING NOT NULL AS (lower(CAST("Col1_4" AS STRING))) VIRTUAL, col1_14 STRING NOT NULL AS (lower("col1̈_ 1")) STORED, col1_15 STRING NOT NULL AS (lower(CAST("cOl1_6" AS STRING))) STORED, PRIMARY KEY ("c'ol1_%p3" DESC, col1_11 DESC, "col1̈_ 1" DESC, col1_2, col1_12 ASC, col1_9 DESC, "c😙ol̟1_%10", col1_13 DESC, "cOl1_6", "col1 _5" DESC, "Col1_4" DESC), UNIQUE ("Col1_4" ASC, (CASE WHEN "col1 _5" IS NULL THEN '':::STRING ELSE 'eOSC':::STRING END) DESC) STORING (c͈ol1_0, col1😪_8, col1_14, col1_15), INDEX ("col1 _5" DESC, col1_12 DESC, "cOl1_6" ASC, col1_14 ASC, "c%vo\\x1bl1_\\x117", c͈ol1_0 DESC, col1_13 DESC, col1_11, "c😙ol̟1_%10", col1😪_8 ASC, "col1̈_ 1" DESC, col1_15 ASC, col1_2, "c'ol1_%p3" ASC, lower(CAST("Col1_4" AS STRING)) ASC), UNIQUE ((CASE WHEN "col1 _5" IS NULL THEN e'{\x01\x1d*2N':::STRING ELSE e'\x16^;,p':::STRING END), "c%vo\\x1bl1_\\x117" ASC), INDEX ("c😙ol̟1_%10" ASC, "col1̈_ 1" ASC, col1_11 ASC, col1_13) PARTITION BY LIST ("c😙ol̟1_%10", "col1̈_ 1", col1_11) (PARTITION "table1_p�ar""t0" VALUES IN (('-1 years -9 mons -388 days -16:39:27.502288':::INTERVAL, e'{78\x183?':::STRING, 'PI':::STRING), ('1 day':::INTERVAL, e'\'':::STRING, 'td':::STRING), ('67 years 4 mons 382 days 02:50:59.681011':::INTERVAL, e'^$Q \b\x16\x12':::STRING, 'zp[iL}':::STRING)), PARTITION "tAble1_part1" VALUES IN (('31 years 6 mons 937 days 13:39:20.734826':::INTERVAL, e'\x01':::STRING, e'\b':::STRING), ('-22 years -8 mons -55 days -13:11:55.617878':::INTERVAL, e'tHf>!u\n\x01z':::STRING, '':::STRING), ('-1 mons -538 days -03:49:47.469561':::INTERVAL, '"':::STRING, e'-"RJ*``\'':::STRING)), PARTITION table1_part2 VALUES IN (('-32 years -431 days -03:59:25.064881':::INTERVAL, '+dKc':::STRING, '':::STRING), ('-74 years -11 mons -52 days -20:03:06.222104':::INTERVAL, e'\x17\x0fG':::STRING, '':::STRING), ('-64 years -8 mons -879 days -15:18:28.705564':::INTERVAL, e'\x19\x10{)Ye':::STRING, e'\x1dQ':::STRING)), PARTITION """table̱1__part3" VALUES IN (('-37 years -10 mons -560 days -03:37:04.202913':::INTERVAL, '':::STRING, '':::STRING), ('-32 years -7 mons -741 days -21:57:43.764477':::INTERVAL, 'X':::STRING, e'\f\x0f':::STRING), ('-21 years -6 mons -103 days -24:20:35.535448':::INTERVAL, e'+\x18Pu\x11\x1cv3+':::STRING, '':::STRING)), PARTITION "t%fcable1_%vpaRt4" VALUES IN (('3 years 7 mons 297 days 24:38:13.111594':::INTERVAL, e'\x01':::STRING, 't|4':::STRING), ('58 years 4 mons 884 days 22:17:50.341682':::INTERVAL, e'L\\(\x16$k':::STRING, e'/1(7h0\x0f':::STRING), ('10 years 5 mons 50 days 17:39:07.812858':::INTERVAL, 'R':::STRING, 'X':::STRING)), PARTITION table1_part5 VALUES IN (('11 years 1 mon 502 days 20:00:29.136473':::INTERVAL, '8&hOK':::STRING, 'Tmn':::STRING), ('36 years 3 mons 250 days 12:17:49.568752':::INTERVAL, e'Qp\x1fI|yb\x13':::STRING, e'\x19':::STRING), ('-38 years -315 days -04:54:11.715166':::INTERVAL, '_Ik':::STRING, e'tZ\x17\x0e':::STRING)), PARTITION table1_part6 VALUES IN (('-69 years -6 mons -654 days -20:20:07.575846':::INTERVAL, e'(\r\x02\x03<q\x05':::STRING, e'd\x01303':::STRING), ('-1 years -3 mons -575 days -17:47:56.372991':::INTERVAL, e'R\x19\x01iwcn':::STRING, e'\'':::STRING), ('-40 years -2 mons -489 days -23:34:51.049431':::INTERVAL, e'Y*\x05*Pd#nz':::STRING, e'{ZV\bVqyK':::STRING)), PARTITION "DEFAULT" VALUES IN ((DEFAULT, DEFAULT, DEFAULT))), INDEX (lower(CAST("c'ol1_%p3" AS STRING)) DESC) STORING (c͈ol1_0, "c%vo\\x1bl1_\\x117", col1_14, col1_15), INDEX ("c%vo\\x1bl1_\\x117" DESC, (col1_2 + 24917:::INT8) DESC, col1_14, c͈ol1_0 ASC, "col1̈_ 1", col1_11 ASC, "c'ol1_%p3", "cOl1_6" DESC, col1_2, col1_9) WHERE ((((((table1.col1_14 >= '':::STRING) AND (table1.col1_12 > (-9223372036854775807):::INT8)) OR (table1.col1😪_8 < '3000-01-01 00:00:00+00':::TIMESTAMPTZ)) AND (table1.col1_2 > 0:::INT8)) OR (table1.col1_11 >= e'\U00002603':::STRING)) OR (table1."col1̈_ 1" > e'\U00002603':::STRING)) AND (table1.col1_13 >= 'X':::STRING), FAMILY ("cOl1_6", "c'ol1_%p3", "Col1_4"), FAMILY ("col1 _5", col1😪_8), FAMILY (col1_14), FAMILY (col1_2, col1_11, col1_15, col1_9), FAMILY ("c%vo\\x1bl1_\\x117", c͈ol1_0), FAMILY ("col1̈_ 1"), FAMILY (col1_12), FAMILY ("c😙ol̟1_%10"));
    rsg_test.go:697: ALTER TABLE table1 INJECT STATISTICS e'[{"avg_size": 30, "columns": ["\\"cOl1_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 85106133155, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 28, "columns": ["col1\U0001F62A_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55598977134, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 27, "columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48412369477, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 16, "columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66838873941, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 9, "columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57167914929, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 5, "columns": ["\\"c%vo\\\\\\\\x1bl1_\\\\\\\\x117\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58235694914, "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 4, "columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2514748677, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 28, "columns": ["c\u0348ol1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 87318031840, "histo_col_type": "", "name": "__auto__", "null_count": 61789620589, "row_count": 90000000000}, {"avg_size": 26, "columns": ["\\"col1\u0308_ 1\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 89846547068, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 12, "columns": ["\\"c\'ol1_%p3\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29810749152, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "2F7753EA/F3FFC04A"}, {"distinct_range": 2217482639211513600, "num_eq": 700000, "num_range": 5044373563750955660, "upper_bound": "5A36672B/E694BE42"}, {"distinct_range": 0, "num_eq": 700, "num_range": 4416179762673237912, "upper_bound": "7FFFFFFF/FFFFFFFF"}, {"distinct_range": 60000000000, "num_eq": 10000, "num_range": 60000000000, "upper_bound": "A23B1418/8B10215B"}, {"distinct_range": 86757.1425971595, "num_eq": 7946033089942322471, "num_range": 100000, "upper_bound": "CFB64CB1/6B2C2912"}, {"distinct_range": 890933332661555300, "num_eq": 5073305387328883618, "num_range": 1064740806404889665, "upper_bound": "DD18AB8F/4A0A6EB"}, {"distinct_range": 2977332761779017700, "num_eq": 9000000, "num_range": 2977332761779017953, "upper_bound": "F06721D9/187BEB3A"}], "histo_col_type": "PG_LSN", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 24, "columns": ["\\"Col1_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44319566009, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 8, "columns": ["\\"col1 _5\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53032282285, "histo_buckets": [{"distinct_range": 0, "num_eq": 1140595257531203324, "num_range": 0, "upper_bound": "256615090"}, {"distinct_range": 4000, "num_eq": 9221242816186231424, "num_range": 4000, "upper_bound": "3758885987"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 2, "columns": ["\\"c\U0001F619ol\u031F1_%10\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17361688447, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 20, "columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63632492156, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 26, "columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44623511497, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"avg_size": 23, "columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58183706366, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}]':::JSONB;
    rsg_test.go:697: CREATE TYPE rand_typ_0 AS ENUM ('hpoy', 'kht', 'l');
    rsg_test.go:697: CREATE TYPE rand_typ_1 AS ENUM ('lvm', 'l', 'eetcqx', 'sw');
    rsg_test.go:697: CREATE TYPE rand_typ_2 AS ENUM ('zz', 'u', 'p', 'v', 'vq', 'yslfy');
    rsg_test.go:697: SET sql_safe_updates = false;;
    rsg_test.go:697: 
        BEGIN; CREATE TYPE greeting AS ENUM ('hello', 'howdy', 'hi', 'good day', 'morning'); COMMIT;
        BEGIN;
        CREATE TABLE IF NOT EXISTS seed AS
        	SELECT
        		g::INT2 AS _int2,
        		g::INT4 AS _int4,
        		g::INT8 AS _int8,
        		g::FLOAT4 AS _float4,
        		g::FLOAT8 AS _float8,
        		'2001-01-01'::DATE + g AS _date,
        		'2001-01-01'::TIMESTAMP + g * '1 day'::INTERVAL AS _timestamp,
        		'2001-01-01'::TIMESTAMPTZ + g * '1 day'::INTERVAL AS _timestamptz,
        		g * '1 day'::INTERVAL AS _interval,
        		g % 2 = 1 AS _bool,
        		g::DECIMAL AS _decimal,
        		g::STRING AS _string,
        		g::STRING::BYTES AS _bytes,
        		substring('00000000-0000-0000-0000-' || g::STRING || '00000000000', 1, 36)::UUID AS _uuid,
        		'0.0.0.0'::INET + g AS _inet,
        		g::STRING::JSONB AS _jsonb,
        		enum_range('hello'::greeting)[g] as _enum
        	FROM
        		generate_series(1, 5) AS g;
        COMMIT;
        
        INSERT INTO seed DEFAULT VALUES;
        CREATE INDEX on seed (_int8, _float8, _date);
        CREATE INVERTED INDEX on seed (_jsonb);
        ;
    rsg_test.go:697: ALTER TABLE seed SPLIT AT VALUES (8904123274845647528);
    rsg_test.go:697: ALTER TABLE seed SPLIT AT VALUES (145562922025433792);
    rsg_test.go:697: ALTER TABLE seed SPLIT AT VALUES (5873368786789168339);
    rsg_test.go:697: ALTER TABLE seed SPLIT AT VALUES (305061984578372166);
    rsg_test.go:697: ALTER TABLE seed SPLIT AT VALUES (343939023556521163);
    rsg_test.go:697: ALTER TABLE seed SPLIT AT VALUES (5076132052420508564);
    rsg_test.go:697: ALTER TABLE seed SPLIT AT VALUES (3412737986666417211);
    rsg_test.go:697: ALTER TABLE seed SPLIT AT VALUES (1038813455422250749);
    rsg_test.go:697: ALTER TABLE seed SPLIT AT VALUES (5233757564500418459);
    rsg_test.go:697: ALTER TABLE seed SCATTER;
    rsg_test.go:699: 
    rsg_test.go:700: -- test log scope end --
--- FAIL: TestRandomSyntaxSQLSmith (303.79s)
test logs left over in: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxSQLSmith3552869484
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxSQLSmith.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/internal/sqlsmith/relational.go
+++ b/pkg/internal/sqlsmith/relational.go
@@ -109,10 +109,10 @@ var (
 		{1, makeRestore},
 		{1, makeExport},
 		{1, makeImport},
+		{1, makeCreateFunc},
 	}
 	nonMutatingStatements = []statementWeight{
 		{10, makeSelect},
-		{1, makeCreateFunc},
 	}
 	allStatements = append(mutatingStatements, nonMutatingStatements...)
 

